\newpage{}

# Is there evidence of diploid-tetraploid hybridisation in a *Euphrasia* contact zone?

I acknowledge that Hannes Becher carried out demographic simulations and assisted in the interpretation of the genetic data, and Sebastian Williams generated all ITS1 sequence data.

## Abstract

Cross-ploidy hybridisation is expected to be rare due to the strong ploidal prezygotic barrier. Hybridisation between species that differ in ploidy level however, has been reported across flowering plants. British eyebrights (*Euphrasia*) represent a good study system to investigate cross-ploidy hybridisation, as 71 different hybrid combinations have been reported and at least two putative hybrid species (*E. vigursii* and *E. rivularis*) have formed from progenitor species of different ploidy levels. In this study, we analysed a contact zone between the diploid species *E. rostkoviana* and the tetraploid species *E. arctica* in Wales. We sequenced the internal transcribed spacer region (ITS1), and used Genotyping by Sequencing (GBS) to look for evidence of cross-ploidy hybridisation and introgression. All sites in the ITS1 region were fixed between diploids and tetraploids, indicating a strong barrier to hybridisation. Further, analysis of the GBS data using PCA, STRUCTURE, and AMOVA across 270 SNPs, indicated clear separation between the ploidy levels. While the global F~ST~ between species was high at 0.44, the distribution across all SNPs was bimodal, indicating potential differential selection on loci between diploids and tetraploids. Only using demographic inference with $\delta{a}\delta{I}$ did we find evidence of limited gene flow – around one or fewer migrants per generation. Overall, our results are consistent with cross-ploidy hybridisation being rare or absent. Overall, secondary contact between species of different ploidies can result in a mix of outcomes - from hybrid speciation, to rare hybridisation as reported here.

## Introduction

Natural hybridisation is an important evolutionary phenomenon with wide ranging consequences, from extinction [@RN935], to hybrid speciation [@RN998]. Most studies to date have investigated hybridisation between diploid species, while hybridisation between species that differ in their ploidy level (cross-ploidy hybridisation) has generally received less attention [though see @RN1115]. While this may in part be due to technical issues with inferring homology relationships between diploids and polyploids, there are also clear biological reasons. Contrasting ploidy levels represent a known, highly effective barrier to hybridisation [@RN1018]. The main barriers are abnormal endosperm ratios of maternal:paternal genomes at fertilisation which prevent hybrid seed formation [@RN1419], and later hybrid sterility caused by irregularities in chromosome pairing at meiosis leading to aneuploid gametes [@RN1298]. Both of these factors prevent hybridisation and introgression between species with contrasting ploidy levels. These barriers can be overcome through unreduced gamete production in the lower ploidy parent, or where a triploid (or other intermediate ploidy) F1 hybrid is formed, by either backcrossing to one of the parental species [@RN918], or by whole genome duplication to restore fertility [@RN311]. Cross ploidy hybridisation may be an important as a mechanism for maintaining genetic variation in polyploid species [although distinguishing this from recurrent polyploidisation can be difficult; @RN1389], exchanging adaptive alleles between species [@RN980], and has been shown to generate new polyploid cytotypes or species [@RN311].

A number of natural cross-ploidy hybridisation examples exist in the literature and these cover a variety of phylogenetically distinct taxa. For example, there is cross ploidy hybridisation reported in *Dactylorhiza* [@RN1067], *Mercurialis* [@RN1056], and *Epidendrum* [@RN905], the latter two of which often form hybrid zones. Where hybridisation is particularly common, hybrid swarms can develop, as seen in co-occurring diploid and tetraploid species of *Cochlearia* [@RN1395]. Cross ploidy hybridisation in the genera *Senecio* and *Mimulus* have led to the creation of three hybrid species endemic to Britain [@RN311; @RN1383]. Two of these hybrid species have resulted from whole genome duplication of initial triploid F1 hybrids (*Senecio cambrensis* and *Mimulus peregrinus*), whilst the other species was created through introgression to the tetraploid parent (*Senecio eboracensis*). Although far from exhaustive, these examples highlight that cross-ploidy hybridisation involves mostly diploid and tetraploid species and introgression, when it occurs, is usually in the direction of the higher ploidy parental species. Here, the higher likelihood of unreduced gametes produced by the lower ploidy parent may allow it to form fertile lineages with the higher ploidy parent [@RN1020; @RN1380].

*Euphrasia* (Orobanchaceae) is a large temperate genus of hemiparasitic plants, with around 263 species worldwide (Nickrent pers. comm.). In Britain and Ireland, there are 21 species of *Euphrasia*, which and are considered a taxonomically complex group characterised by recent postglacial divergence [@RN1326], plastic phenotypes [@RN1225], and the widespread occurrence of natural hybridisation [@RN1156]. The genus in Britain and Ireland consists of five diploid and sixteen tetraploid species. This ploidy difference is associated with contrasting mating systems where the tetraploids are mixed maters, or highly selfing, while the diploids outcross more extensively [@RN375]. The tetraploid species are allotetraploids, with two subgenomes that are around 5% divergent. One subgenome of the tetraploid species is only 0.2% divergent from the diploid species genome, making it likely that the British diploid species are one of the parents of British tetraploid species [@RN1413]. Out of 72 hybrid *Euphrasia* combinations reported in the British flora, 13 are reported to be diploid-tetraploid hybrids based on morphology [@RN1007]. Unusually, these cross ploidy hybrids are purported to be diploids derived from triploid F1s that backcross to the diploid parent [@RN423]. This contrasts with most prior predictions as to the directionality of cross ploidy hybridisation, where the hybrids are tetraploids. In the proposed scenario, genetic material from the tetraploid species is expected to introgress into the diploid species through backcrossing to the diploid parent. The diploid-like subgenome of the tetraploid species is homologous to extant diploid species and therefore expected to successfully pair and recombine with diploid chromosomes. 

Here, we present a genetic analysis of a contact zone between diploid *E. rostkoviana* and tetraploid *E. arctica*. Both of these species have large corollas suggestive of higher rates  outcrossing [@RN1156], and hybrids have been identified widely across Britain [@RN1007], however putative F1 triploid hybrids are yet to be found in the wild. We look for any evidence of hybridisation and introgression between ploidy levels. First, we used Sanger sequencing of a locus that shows diagnostic differences between diploids and tetraploids to see if there is evidence of hybridisation. We then used Genotyping by Sequencing (GBS) to look for evidence of introgression. We also used demographic modelling to investigate the most likely scenario of historical gene flow to explain the observed genetic structure. The results enable us to discuss the processes governing reproductive isolation in a diploid-tetraploid contact zone. We predict that as reproductive barriers in *Euphrasia* are low, and as cross ploidy hybridisation is being increasingly found in other plant groups using genomic data, that there will be cross ploidy hybrids and evidence for introgression.

## Materials and methods

### Population sampling and DNA extraction

Population sampling took place in July 2017, at the managed hay meadow at Cae Trawscoed in the National Botanical Garden Wales (lat/long: 51.8447/-4.14531) where there is a mixed population of diploid *Euphrasia rostkoviana* and tetraploid *Euphrasia arctica*. These species can be easily separated by morphology based on the presence or absence of long-stalked flexuous glandular hairs. Putative hybrids show a low density of long glandular hairs and are intermediate for a range of other traits. Both species were abundant and distributed amongst each other, with *E. rostkoviana* present amongst taller vegetation, and *E. arctica* more dominant in shorter cropped vegetation. A total of 95 individuals were sampled from the mixed population, 45 being identified based on morphology as diploid and 50 being tetraploid. Plants were sampled evenly along transect of approximately 12m where plants were highly intermixed. Specimens were transferred into silica bags to dry until DNA extraction. DNA was extracted using the DNeasy Plant Mini kit (Qiagen), following manufacturer’s protocols. 

### Sanger sequencing and sequence analysis

ITS sequencing was used as there is a known diagnostic difference between diploid and tetraploid species [@RN1326], while plastid sequencing used the most variable locus that is widely used in population genetic studies of *Euphrasia*. The sample size for the ITS sequencing was 70 individuals, while for plastid sequencing eight individuals were used. PCRs were performed in 25$\mu$L reactions; DNA amplification protocols and conditions for PCRs are given in Appendix 3 Tables 5 and 6.  To check the quality of PCR products, the DNA was visualised on a 1% agarose gel. PCR products were then cleaned with Exo-SAP (Affometrix) using standard protocols and submitted to Edinburgh Genomics for sequencing reactions using BigDye Terminator Cycle Sequencing chemistry and Sanger Sequencing on an ABI 3730. ITS1 PCR products were sequenced in the forwards direction only (with the ITS4 primer), whilst *rpL32*-*trnL*~UAG~ was sequenced in both directions.

ITS1 and *trnL* spacer chromatograms were aligned and edited in Geneious [version 9.0.5; @kearse2012geneious]. Low quality bases were trimmed at the beginning and ends of sequences; six ITS1 sequenced were excluded due to poor sequence quality. The 558bp *trnL* spacer alignment of eight sequences showed no variable sites and was therefore not analysed further. The final ITS1 alignment included 62 individuals and was 658 bp in length with 58 variable sites in total. 26 diploids and 36 tetraploids were present in the alignment. An outgroup species *E. transmorrisonensis* (diploid; from Taiwan), was added from NCBI (GenBank accession number: AY165615) to polarise the ITS1 phylogeny for visualisation but was not used in downstream analyses. We constructed a Maximum Likelihood phylogeny using IQ-TREE [version 1.5.5; @RN1223] using ModelFinder to find the most suitable substitution model (using the TESTNEWMERGE model flag) with 1000 ultra-fast bootstraps [@RN1306]. The resulting newick file was visualised in ggtree [version 2.1.2; @RN1420]. To further characterise sequence variation, this alignment was read into R version 3.6.1 [@RCoreTeam] using the function read.dna() from the package ape [version 5.4; @RN1421] and converted to a genind object with adegenet [version 2.1.2; @RN1422]. We tested how much genetic variation was partitioned between ploidy levels using Analysis of Molecular Variance (AMOVA) implemented in the poppr [version 2.8.4; @RN1423] function poppr.amova().

### GBS & SNP discovery

GBS library preparation was performed following the protocol in [@RN1121]. We used the enzyme ApeKI to fragment the genome and create cut sizes for adapter ligation. Samples were then pooled and cleaned before PCR amplification and sequencing. A single well was used as a water control. The pooled library was submitted to the University of Oregon for 100bp single end sequencing on the Illumina HiSeq 4000, generating 17,397,350 reads. We then used the TASSEL 5 pipeline version 2 to discover SNPs using default settings, except the k-mer length was increased to 75 [@RN1424; see https://github.com/Euphrasiologist/GBS_V2_Tassel_5]. Master sequence tags (n = 1,001,272) were aligned to a *Euphrasia arctica* reference genome [@RN1413] using BWA with default settings [version 0.7.17; @RN1425]. All scaffolds in the reference genome greater than 1000bp were merged into a single scaffold to reduce computational time. The VCF file was filtered for variants with >50% missing data and individuals with >75% missing data using vcftools [version 0.1.16; @RN1426]. 

We focused our analyses of genetic structure, hybridisation and introgression on regions that are homologous between diploids and tetraploids, and not in the other more divergent tetraploid subgenome where gene exchange with diploids is unlikely. As such, variants were excluded if they were not located on the ‘conserved’ set of scaffolds of presumed disomic inheritance that were identified in genome-wide sequence comparisons of diploid and tetraploid *Euphrasia* by @RN1413. The conserved set comprises 46 Mbp (n = 3454) of sequence and is likely to represent (a part of) the conserved subgenome. A minor allele frequency filter was applied to remove invariant sites using vcftools. One variant was kept per scaffold to ensure variants were not tightly linked, using PLINK [version 1.9; @RN1427]. This left 92 individuals (42 diploids and 50 tetraploids) and 270 variable sites for analyses.

### Identifying hybridisation between ploidy levels

To investigate hybridisation between the two species of *Euphrasia* present in the contact zone, we first conducted a Principal Component Analysis (PCA) of our genomic GBS dataset. First, the VCF was loaded into R using the package vcfR [version 1.10; @RN1428] and the PCA was carried out using the adegenet [@RN1422] function dudi.pca() where missing values were substituted by the mean allele frequencies. The PCA was visualised using ggplot2 [version 3.2.1; @Wickham2016a]. Second, we performed a Bayesian admixture analysis in the program STRUCTURE [version 2.3.4; @RN1430] using the same GBS dataset. The VCF was converted to a STRUCTURE file format using PGDSpider [version 2.1.1.5; @RN1435]. We set the K-value to be 2 as there were two divergent species of differing ploidies, and the run was set with a burn-in of 100,000 for 1,000,000 iterations on the ‘admixture’ option. In addition, 90% probability intervals were stored using the ANCESTPINT option. Samples where probability intervals overlapped zero or one were considered non-hybrids. The Q-matrix, and probability intervals, were extracted from the output and plotted using a custom R script (https://github.com/Euphrasiologist/StructuRe). Third, we explicitly attempted to identify hybrid individuals using the program NEWHYBRIDS [version 1.1; @RN928], which classifies individuals into one of six potential categories (parent A, parent B, F1, F2, backcross (BC)1 to parent A, BC1 to parent B) based on their SNP genotypes. The model was run with a burn in of 100,000 iterations and a run length of 100,000 sweeps.

### Quantification of genetic variability within and between ploidy levels

We computed several population genetic statistics on the GBS dataset to understand population structure in the contact zone, and used AMOVA (as above) to detect regions of the genome that may have introgressed. Weir and Cockerham’s estimator of F~ST~ was calculated for each SNP in the GBS data using vcftools [@RN1426] and visualised in ggplot2 [@Wickham2016a]. The average F~ST~ across all SNPs was reported as the global F~ST~. An AMOVA was run for both the ITS1 and GBS datasets using the function poppr.amova() from the R package poppr [@RN1423], which was used to understand the partitioning of genetic variability both within and between ploidy levels. P-values were then derived from the output using the randtest() function from ade4 [version 1.7-15; @RN1431], randomly permuting sample matrices 9999 times. 

### Demographic inference with $\delta{a}\delta{I}$

We carried out demographic model fitting using the package $\delta{a}\delta{I}$ to see what the best fitting model of hybridisation was for the disomic GBS data [@RN1418]. While we acknowledge that a limited number of SNPs were detected, these analyses should be able to tell us the difference between alternative scenarios of gene flow [@hartmann2020higher]. Missing data was handled by scaling down the size of the joint site-frequency spectra to 24 (haploid) genomes per species. We implemented four models, each involving one ancestral population splitting into two sub-populations corresponding to diploids and tetraploids, which could differ in size. Model parameters corresponding to the diploid and tetraploid sub-populations are denoted with the subscripts D and T. The models differ by the amount of gene flow allowed between the sub-populations: (1) constant gene flow with five parameters (two population sizes N~e~D and N~e~T, two migration rates MDT and MTD, and the time of the population split T0), (2) historic gene flow only with six parameters (two population sizes, two migration rates, the time when gene flow ceased T1, and the time difference between T1 and the time of the split denoted T0), (3) secondary contact as (2) but with gene flow in T0 and not in T1, and (4) no gene flow with three parameters (two population sizes and T0, see Figure \ref{CH4F3}A for schematics). We fixed F at 0.75 and 0.81 for *E. arctica* and *E. rostkoviana* according to the empirical estimates. To assess the uncertainty of the model fitting, we used 99 individual sub-samplings of our data set. Because we found the model fitting results to depend strongly on the initial conditions, we ran 99 replicates with randomly perturbed starting values for each model and down-sampled data set resulting in 39,204 optimisations. From each set of 99 replicates, we selected the one with the best log likelihood. In order to compare these nested models with different numbers of parameters, we computed the Akaike Information Criterion (AIC) of each fit, and we plotted the results with matplotlib [@RN1443].

## Results

### Patterns of genetic structure in ITS1

The 658bp alignment of the ITS1 sequence data included 26 diploid individuals and 36 tetraploid individuals. Eight sequences were removed due to low sequence quality. The chromatograms revealed no evidence of sequence additivity or double peaks, which might have indicated hybrid individuals or retained duplicate copies in the polyploids. The maximum likelihood phylogenetic tree showed two distinct clades of *Euphrasia* that were highly supported (Figure \ref{CH4F1}A). All 58 SNPs in the alignment were fixed between the ploidy groups. Accordingly, an analysis of molecular variance (AMOVA) showed that 99.5% of the ITS1 variation in the samples was explained by ploidy (p < 0.001; Table \ref{CH4T1}A). Limited sampling of four individuals from each species for the *trnL* spacer showed no differences between the species.

![ITS1 marker and GBS analysis suggest strong barriers to gene flow between diploid (black) and tetraploid (orange) species of *Euphrasia*. A) Phylogenetic tree of the ITS1 marker generated from IQ-TREE; *E. transmorrisonensis* is used as an outgroup species. Bootstrap support is visualised on a colour scaled point at each node and only support values above 75 are shown. B) A Principal Component Analysis of 92 *Euphrasia* individuals across 270 loci located on different scaffolds of the reference *E. arctica* genome. The first two Principal Components are plotted against each other. C) Admixture analysis performed in STRUCTURE on the same 92 individuals, where K = 2.\label{CH4F1}](./PDFs/Figure_1_updated.pdf){ width=100% }

\newpage{}

### Patterns of genetic structure from GBS data

Analysis of SNP data generated using GBS corroborated the findings from the ITS1 dataset. In total, 270 SNPs were analysed representing putatively disomically inherited scaffolds in the *Euphrasia arctica* reference genome shared between diploids and tetraploids (see Methods). Principal Component analysis (PCA) showed that the first principal component explained 32.9% of the genomic variation and clearly separated individuals by ploidy, with two separate clusters and no evidence of intermediate genotypes (Figure \ref{CH4F1}B). The same pattern was also present in the STRUCTURE analysis with a K value of 2, which reported Q values all above 0.98 (Figure \ref{CH4F1}C) that assigned individuals to clusters consistent with their morphological identification. The probability interval of each Q value overlapped either zero or one, indicating no hybrid genotypes. An AMOVA on the GBS data showed 78.4% of genomic variation was explained by ploidy (p < 0.001; Table \ref{CH4T1}B). The remainder of the genomic variation (21.6%) was due to differences within ploidy level.

\newpage{}

| Data set | Variation      | DF  | Sum Sq | % Var | p-Monte Carlo |
| -------- | -------------- | --- | ------ | ----- | ------------- |
| A) ITS1  | Between ploidy | 1   | 454.9  | 99.5  | 0.0001        |
|          | Within ploidy  | 60  | 4.9    | 0.5   |
|          | Total          | 61  | 459.8  | 100   |
| B) GBS   | Between ploidy | 1   | 56     | 78.4  | 0.0001        |
|          | Within ploidy  | 90  | 30.3   | 21.6  |
|          | Total          | 91  | 86.3   | 100   |

Table: Hierarchical analysis of molecular variance (AMOVA) for both ITS1 and GBS data sets. The total variation is partitioned between and within ploidy level of individuals in the analysis. Degrees of freedom (df), the variance of each of the observations (Sum sq), and percentage of variation explained by each level of variation (% Var) are reported. The significance of the components of variance are reported as p-values. \label{CH4T1}

### Detecting genomic hybridisation

We calculated population genetic parameters from GBS data to further investigate potential hybridisation between the two species differing in ploidy level. The global F~ST~ between species was high at 0.44, indicating that diploid *E. rostkoviana* and tetraploid *E. arctica* were highly differentiated. The distribution of F~ST~ however showed a bimodal distribution with high count of SNPs that were either mostly shared or private (Figure \ref{CH4F2}). The NEWHYBRIDS analysis assigned each individual as 100% either parental species, with no evidence of F1 hybrids or backcrossed individuals. Altogether, the AMOVA, PCA, STRUCTURE, and NEWHYBRIDS results showed a clear signal of strong differentiation between the two species.

![The distribution of F~ST~ for each of the 270 SNPs in the GBS dataset across all 92 individuals between the two ploidy levels. The red dashed vertical line indicates the mean global F~ST~ across all SNPs (0.44).\label{CH4F2}](./PDFs/wac_fst.pdf){ width=80% }

### Demographic modelling of gene flow

Demographic model fitting with $\delta{a}\delta{I}$ resulted in the highest overall support for models with constant gene flow and with secondary contact (Figure \ref{CH4F3}B). These models consistently scored low AIC values. The difference in AIC between the best model (constant gene flow, median AIC 425.77) and the runner-up (secondary contact, median AIC 425.89) was not significant (t-test t = -1.38, p = 0.17). The other two models had significantly higher median AICs than the best model (historic gene flow: 462.83, t = -7.29, p < 0.001; no gene flow: 452.05, t = -52.31, p < 0.001). However, for some realisations of the data re-sampling, the model with no gene flow scored the best AIC coinciding with generally low estimates of the age of the population divergence time (T0 tends to be lower without gene flow than in alternative models, see Appendix 3 Figure 7). Concrete conclusions are limited due to the low number of SNPs detected, and resulting low statistical power.

The levels of gene flow fitted to our models tend to be low with M, the number of migrants per generation, of the order of 1 or less. The mean values over all replicates for the constant gene flow model were 0.3 (diploid $\rightarrow$ tetraploid, sd = 0.09) and 0.4 (tetraploid $\rightarrow$ diploid, sd = 0.13). For the model with secondary contact the migration rates were fitted to be 1.3 (diploid $\rightarrow$ tetraploid, sd = 1.02) and 0.3 (tetraploid $\rightarrow$ diploid, sd = 0.09). In all models, the effective population size tends to be slightly higher in the diploids than in the tetraploids (Appendix 3 Figure 7, left-hand panels). While our data suggest the presence of gene flow, the similar AIC values we obtained under different models show how different demographic scenarios may produce similar genetic patterns.

![A) Schematics of the demographic models fitted indicating their parameters. Ne, the effective population size, is scaled to be 1 in the ancestral population. $N_{e(D)}$ and $N_{e(T)}$ indicate the ratios of the present-day effective population sizes of the diploid and tetraploid populations respectively, relative to the ancestral one. T is the number of generations to coalescence, which is subdivided in two epochs in the models with ancestral gene flow and secondary contact. M is the number of migrants per generation with subscripts indicating the direction of gene flow. B) The distributions of AIC values for model fits to 99 sub-sampled data sets. The models with constant gene flow and secondary contact are not distinguishable. Vertical lines indicate the distribution medians. \label{CH4F3}](./PDFs/dadi_fits.pdf){ width=80% }

## Discussion

In this study, we investigated hybridisation between diploid *E. rostkoviana* and tetraploid *E. arctica* in a contact zone in south Wales. Neither ITS1, which is known to have diagnostic differences between species, nor the GBS dataset, provide evidence for recent hybridisation or introgression between these two species of *Euphrasia*. This contrasts with the extensive hybridisation known to occur between *Euphrasia* species of the same ploidy level [@RN1156; @RN1007]. For example, using GBS data, @RN1119 showed that there is extensive gene flow from native to invasive tetraploid species of *Euphrasia* in North America. Although we expected to find cross ploidy hybrids, to date there has only been a single wild triploid hybrid *Euphrasia* individual that has been found [@RN423], and previous attempts to synthesise cross ploidy hybrids artificially have been unsuccessful [@RN930]. This result is consistent with ploidy representing a strong reproductive barrier between species [@RN1018]. While we are confident that recent hybridisation is rare (or absent), based on the small sample of individuals we cannot exclude the possibility that early generation hybrids may have been overlooked if they are rare.

Although conventional population genetic analyses show little evidence of recent gene flow between the two *Euphrasia* species in this study, demographic modelling using $\delta{a}\delta{I}$, and inspection of the distribution of F~ST~ values across all SNPs indicates that there may be limited ongoing gene flow. $\delta{a}\delta{I}$ gives low estimates for bidirectional gene flow, which is known to occur in other cross ploidy hybrid systems (Bleeker, 2003). In particular, the higher estimate for tetraploid to diploid gene flow in the constant gene flow model may give support to the hypothesis that Yeo outlined more than 60 years ago [@RN423], and corroborates a recent study on British *Euphrasia* which suggested possible diploid-tetraploid gene flow [@RN1413]. We note again howevevr, the low statistical power of the $\delta{a}\delta{I}$ analyses, due to a limited sample of SNPs. Although the signature of gene flow is low this may be because it is limited to specific regions of the genome. Here, selection may be operating differentially in the diploid and tetraploid species, which may explain the bimodal F~ST~ distribution seen in our dataset (Whitlock, 2008). 

The results from this study are consistent with some cross-ploidy hybrid systems where hybridisation is very rare, or where hybrids are strongly selected against. For example, in diploid *Centaurea pseudophrygia* and tetraploid *C. jacea*, only targeted sampling was able to reveal cross-ploidy hybrids which were otherwise not found by random sampling in 12 different contact zones [@RN1041]. In a contact zone of diploid *Senecio madagascariensis* and tetraploid *S. pinnatifolius* no hybrids have been detected in the field, however genetic analysis of the seeds revealed hybrid genotypes [@RN1440]. It is possible in the *Euphrasia* contact zone that hybrid seed is being formed but either does not germinate, or hybrid seedlings do not survive to maturity. Our results stand in strong contrast to those found in *Dactylorhiza*, where triploid F1 cross ploidy hybrids, and backcrossed individuals are found frequently [@RN1067; @RN1441]. The mechanisms underlying these differences in the frequency of cross ploidy hybrids in divergent species are yet to be fully established, and are likely an area for fruitful research.
