\newpage{}

# The emerging importance of cross-ploidy hybridisation and introgression

## Abstract

Ploidy differences are often considered a strong barrier to hybridisation and introgression because viable hybrid zygotes are rarely formed, and if they are formed, they usually produce sterile organisms. The increasing accumulation of cytogenetic, genetic, and genomic datasets however, show that ploidy differences may be a more permeable barrier than previously thought. Cross-ploidy hybridisation is most common in plants, with a third of all hybrids in the British flora forming from parental species of differing ploidy level, and dozens of studies have confirmed the existence of cross-ploidy hybridisation across plants with molecular markers. Cross-ploidy hybridisation is also present in certain animal lineages that are characterised by relatively high levels of polyploidy, and where hybrid derivatives are hybridogenetic and do not recombine. Groups with cross-ploidy hybrids often involve allopolyploidy, where homologous chromosomes may pair regularly with a related diploid, and show directional introgression towards the higher ploidy parent. Cross-ploidy hybridisation may lead to introgression of adaptive loci, and be an important driver of diversification and speciation in plants and animals.

## Introduction

Climate change, habitat disturbance and large-scale translocations of species resulting from human activities are increasing contacts between species previously isolated by geographical and ecological barriers, thus raising their potential to hybridise [@RN1465; @RN1466; @RN1467]. There are a range of barriers to cross species hybridisation however, from prezygotic barriers which affect the formation of a zygote (e.g. differing flowering times), to postzygotic barriers that affect the fertility of the hybrid organism (e.g. uneven chromosome complements). Closely related species isolated by prezygotic barriers are more likely to hybridise [@RN902]; however, even species isolated by very strong postzygotic barriers do hybridise in some instances. In plants, polyploidy is a strong reproductive barrier affecting both prezygotic and postzygotic isolation [@RN1115]. Further, as polyploidy is so common in plants, its effect as a reproductive barrier is expected to be widespread and common. 

Polyploidy (see Glossary; Box 1), which is particularly common in plants, creates a very strong postzygotic barrier between species that differ in ploidy (Box 2). For example, many crosses between diploid and tetraploid species either fail to produce a viable zygote or produce poorly formed ones, depending on the direction of the cross. This phenomenon is known as ‘triploid block’ [@RN918]. Should a triploid hybrid form, it is normally either completely or partially sterile, due to formation of malfunctioning gametes containing unbalanced chromosome numbers. On occasion, however, some species differing in ploidy do produce hybrid offspring, triggering gene exchange or possibly the origin of new species via allopolyploidy (Box 3). The importance of such events is not to be underestimated; for example, they have led to the origin of some very recently originated plant species, which are now models for the study of polyploid speciation [@RN902], and also to the origin of some of our most important crop plants, including wheat, sweet potato and sugar cane. Nonetheless, the frequency of cross-ploidy (or interploidy) hybridisation in the wild is a neglected topic, with information related to it scattered through the literature. Here, we bring this information together and emphasise its biological significance.

---------------------------------------------------------------------------------
Box 1: Polyploidy                                               
---------------------------------------------------------------------------------
Polyploidy is the condition where a cell contains more than two sets of chromosomes as a result of whole genome duplication (WGD). The two major routes to polyploidy are either through WGD of a single species chromosome complement, known as autopolyploidy or through hybridisation between two species followed by WGD, known as allopolyploidy [@RN918]. It is driven especially by the production of unreduced gametes in diploid species [@RN1350] and this is affected by a range of factors including specific genes [@RN1351] and environmental stresses [@RN1189]. Although worldwide, the majority of plant species are diploid [~67%, @RN1189], extensive variability in ploidy levels exist at all taxonomic levels and scales [@RN1344; @RN1343]. Both the spatial and phylogenetic distribution of ploidy variation are unlikely to be uniform however, due to climatic and clade specific effects on unreduced gamete formation [@RN1171; @RN1400; @RN1189]. Historically, autopolyploidy has been regarded as both less frequent and less important in an evolutionary context, than allopolyploidy [@RN1343]. The current wealth of cytological data suggests, however, that at least 10% of species are autopolyploids, with allopolyploids estimated to be at least as frequent [@RN1344; @RN1343; @RN1345]. Allopolyploids have received more attention as they are mainly distinctive morphological taxa described as species, while autopolyploids are often morphologically cryptic and lumped into species complexes [@RN1194; @RN1345]. Polyploidy has been important over evolutionary time in the genesis of new plant and animal lineages [@RN1185], and its signature is imprinted several times over in the genome of every flowering plant [@RN1346]. Both polyploid speciation conferring immediate reproductive isolation [@RN1347] and polytopic origins of polyploids can lead to new lineage formation [@RN1349]. There is also increasing evidence that polyploidy facilitates lineage diversification, though this remains a controversial topic [@RN1187; @RN1437; @RN1436]. Polyploidy is also associated with major shifts in ecology and morphology across a wide variety of plant species [@RN1352; @RN1018; @RN1354].
---------------------------------------------------------------------------------

The first known artificial hybrid from crossing two parents of differing ploidy level was created by Kölreuter in 1761 between diploid *Nicotiana paniculata* and allotetraploid *N. rustica*. This hybrid was known as the first “botanical mule” due to its shrivelled anthers and malformed ovaries, indicative of high sterility [@RN1355]. Further artificial crosses demonstrated the formation of other interploidy hybrids that were partially or completely sterile, but nothing was discovered of the frequency or importance of the phenomenon in the wild until much later [@RN1409]. Beginning around the mid C20th, cytogenetic studies became more frequent and revealed extensive ploidy variation both within and between species that varied with geography, and which could be used to explain evolutionary relationships [@RN1357; @RN1356]. However, it was with the availability of multiple nuclear markers in the 1990s that researchers reliably detected hybridisation and introgression between species of differing ploidy [@RN1414; @RN315]. Now, by examining many thousands of genetic markers across the genomes of target species, there is potential to detect cases of adaptive introgression [@RN1364]. Moreover, by focusing on specific genes, examples are now known of cross-ploidy introgression resulting in the transfer of particular traits that markedly affect the biology and fitness of recipient species [@RN312; @RN980; @RN1380; @RN1468]. 

While there have been many recent reviews on the mechanisms that underlie polyploidy and the prevalence of polyploids in nature [e.g. @RN1359; @RN1287; @RN1360; @RN1361; @RN1363], and on the importance of natural hybridisation [e.g. @RN907; @RN1364; @RN1001; @RN1008], our aim is to reconcile early work on cytological variation with recent work on genomics, to consider whether cross-ploidy hybridisation between species may be more prevalent and important than previously known. We first review the presence of cross-ploidy hybridisation in the British and Irish flora, the most well-studied, large-scale flora examined to date. Next, we review the prevalence of cross-ploidy hybridisation inferred with genetic markers that has been reported in the literature, and highlight some general patterns. Lastly, we explore the biology of cross-ploidy hybrids, and discuss how advances in sequencing technology may aid hybrid detection to assess more accurately the state of interploidy hybridisation in nature. We emphasise case studies in flowering plants, where hybridisation and polyploidy are particularly prevalent and well-documented, but also consider other organismal groups where cross-ploidy hybridisation may occur.

## Occurrence of natural cross-ploidy hybrids

Of major interest is how common cross-ploidy hybrids are in nature given the varied constraints of both pre and postzygotic isolation in their generation (Box 2). The evidence required to prove interploidy hybridisation is confirmation of parental ploidy differences, which may come from chromosome counts [@RN1195], genome size estimates (Plant C-value Database) or genomic information [@RN1471], and evidence of hybridisation, which may be from genetic data or from other sources such as morphology [@RN1472]; though see issues with using morphological data to detect hybrids below. Data on both ploidy and hybridisation are patchy, and this limits our current understanding of the frequency of cross-ploidy hybrids in nature. To illustrate the extent of cross-ploidy hybridisation, we consider the case of the British and Irish flora, which contains a manageable number of native species [~1500, excluding large taxonomically complex groups; @RN1309], and is exceptional in having near complete information on species chromosome counts (BSBI Cytology Databse), and the extent of natural hybridity [@RN1007]. Most of the 1295 species for which there is detailed ploidy information are diploids (56%), with higher ploidy levels becoming exponentially less common (Figure \ref{CH2F1}). Between families, however, the distribution of ploidy levels changes significantly, which alters the raw material for cross-ploidy hybridisation to act on (Appendix 1 Figure 1). In terms of hybridisation, there are 909 known hybrids present in the flora [@RN1007]. Of the 588 hybrids that contain ploidy information (321 hybrids lack appropriate data), 203 interploidy hybrids have formed in Britain and Ireland (35%; Appendix 1 Table 1), in comparison to 385 intraploidy hybrids (65%). Cross-ploidy hybrids occur in 67 genera, with over a quarter present in the genera *Rumex* (Polygonaceae, 24), *Salix* (Salicaceae, 19) and *Euphrasia* (Orobanchaceae, 13; Figure \ref{CH2F2}). The majority (55%) of cross-ploidy hybrids involve diploid-tetraploid crosses, with higher order ploidy crosses closely following (43%), and diploid-triploid crosses in the minority (2%). Same ploidy parental species are 22% more likely to form hybrids than parental species of different ploidies (Chapter 3), indicating that ploidy level represents a considerable barrier to hybrid formation, but is far from complete. In addition for flowering plants surveyed above, cross-ploidy hybridisation is likely to be prevalent in other plant groups, such as ferns and fern allies, due to highly variable ploidies and abundant hybridisation. One dramatic example, inferred based on morphology and habitat, is from the lycopod genus *Isoetes*, where the diploid *Isoetes echinospora* (2n = 22) hybridises with the decaploid *I. lacustris* (2n = 110) to produce a hexaploid hybrid (2n = 66).

![Distribution of ploidy levels across the British and Irish flora. Shown are the number of species at each ploidy level which are not known to have multiple cytotypes. Odd ploidies are less frequent than even ploidies, resulting in a ‘saw tooth’ pattern. The most highly polyploid species is *Leucanthemum maximum* at 22-ploid.\label{CH2F1}](./PDFs/Ploidy_dist.pdf){ width=80% }

---------------------------------------------------------------------------------
Box 2: Ploidy differences as a reproductive barrier
---------------------------------------------------------------------------------
Ploidy differences have often been cited as strong reproductive barriers to hybridisation in plants [@RN1018; @RN1019]. Cross-ploidy hybridisation is therefore usually considered rare because hybrids will have unbalanced chromosome content and therefore irregular pairing of chromosomes, rendering the hybrid infertile. This infertility prevents or limits the formation of backcross hybrids and the potential for introgression. In cross-ploidy hybridisation the usual reproductive barriers to cross species mating apply, along with specific factors associated with ploidy level difference between parental species. In addition to reproductive barriers caused by differences in geography, phenology, morphology and mating system etc. [@RN1047; @RN1367; @RN1368], the ploidy ratio of the pollen:style is important [@RN1289; @RN1370], and following fertilisation is a period where endosperm development and (epi)genetic compatibilities are critical [@RN1371; @RN1063].

There are two main pathways to creation of cross-ploidy hybrids; either through reduced or unreduced gametes. Reduced (“normal”) gametes of the both parental species results in the generation of a hybrid with intermediate ploidy. These hybrids, usually triploids derived from diploid-tetraploid crosses, are common and found in a variety of taxa where congeners co-occur, for example *Aconitum*, *Ficaria*, *Dactylorhiza* and *Senecio* [@RN1169; @RN1167; @RN1067; @RN911]. A barrier to the creation of these hybrids through reduced gametes is known under the umbrella term ‘triploid block’ [@RN1381; @RN1344]. Early work on experimental diploid-autopolyploid crosses established the presence of a triploid block and that direction of crosses was important [@RN1374; @RN1020; @RN1375]. The major cause of triploid block is attributed to genomic conflict in the maternal endosperm, which is usually triploid and composed of a ratio of two maternal and one paternal genomes [@RN1063]. Deviations from this ratio cause the endosperm to malfunction in development and function [@RN1363]. Reciprocal crosses differ in their likelihood of success, and it is a general phenomenon that crosses where the higher ploidy parent is female are more likely to produce viable offspring, due to endosperm ratios which are better tolerated [Figure \ref{CH2BF1} panels a and b; @RN1376]. Triploid block may also be caused by the action of allelic incompatibilities at an early stage in development, although this topic is little explored [@RN1377]. A second possibility in the creation of cross-ploidy hybrids is where the lower ploidy parent produces unreduced (“polyploid”) gametes. Unreduced gamete production is on average 0.1-2%, with rare individuals and hybrids that produce considerably higher frequencies [>85%; @RN1171; @RN1378; @RN1379]. In addition, many different taxa produce unreduced gametes, and their production also varies with environmental variables [@RN1380; @RN1189]. Successful crosses occur more readily when unreduced gametes are produced by the diploid parent, thus restoring the gamete ploidy to that of the higher ploidy parent [Figure \ref{CH2BF1} panel c) @RN1381].

---------------------------------------------------------------------------------

![Potential outcomes of hybridisation between diploid and tetraploid species. In each panel, the top two circles refer to the parental species, the middle two ellipses to the gametes produced from each parent, the bottom left box to the F1 hybrid and the bottom right box to the endosperm. Panels a and b consider hybridisation with reduced gametes and therefore generate triploid hybrids, while panels c and d consider hybridisation where one parent produces unreduced gametes. In particular, panel c illustrates that a fertile polyploid can be generated in a single generation. Figure generated with graphviz [@RN1401].\label{CH2BF1}](./PDFs/Cross_Ploidy_F1.pdf){ width=80% }

Inferring hybridisation from morphology, geography, cytology and limited genetic data, as is the case with many hybrids in the British and Irish flora, will overlook cryptic hybridisation and introgression that can be detected with multiple genetic markers. Moreover, the extent of cross-ploidy hybridisation in this flora is likely to be affected by extensive habitat disturbance and the prevalence of alien taxa. A wider survey of published studies of hybridisation based on multiple genetic markers or strong cytogenetic evidence revealed 43 different parental species combinations from 48 studies resulting in cross-ploidy hybridisation, with such hybrids present in 33 genera from 16 angiosperm families, three fern families, and three animal families (Table \ref{CH2T1}). Diploid-tetraploid crosses are found in 32 of the 43 parental crosses, with the rest being higher ploidy crosses. This confirms that interploidy hybridisation is likely to be much more widespread than is currently appreciated. The taxonomic spread of interploidy hybridisation is especially broad in angiosperms, as evidenced by data both from the British and Irish flora and the wider literature. For example, monocots are well represented (Liliaceae, Orchidaceae, Poaceae), as are basal eudicots (Ranunculaceae, Papaveraceae) and throughout the rest of the phylogenetic tree scattered in the Fabids, Malvids and Superastrids. This distribution indicates interploidy hybridisation is very widespread and potentially abundant throughout the flowering plant phylogeny (Figure \ref{CH2F2}). On the other hand, the conspicuous absence of records from large, diverse families with variable ploidy, such as Rubiaceae potentially indicate a phylogenetic skew in interploidy hybridisation. Cases of such hybridisation are not just phylogenetically but also geographically widespread, with examples reported from across four continents, though tropical regions are poorly represented and most studies report hybridisation in large temperate or cosmopolitan plant families (e.g. Asteraceae and Orchidaceae). In terms of life form, most well-documented cross-ploidy hybrids (with the notable exception of *Euphrasia*) are perennial, a factor which correlates strongly with hybridisation regardless of parental ploidy level [@RN1285].

![Distribution of cross-ploidy hybrids across the British and Irish flora. The number of cross-ploidy (dark bar) and intra-ploidy (light bar) hybrids are shown per family, in the context of family-level phylogenetic relationships from DNA Barcode UK (Jones et al., In Review). Faded family names indicate missing ploidy data, and red family names highlight those families which contain five or more different ploidy levels. Numbers in parentheses are the number of cross-ploidy hybrids formed per family. Phylogeny generated using lwPhylo [@Brown2020b]. \label{CH2F2}](./PDFs/tree_plot.pdf){ width=80% }

In contrast to flowering plants, polyploidy in animals and fungi is thought to be rare, famously so in mammals and birds, though many examples are known in certain lineages of amphibians, teleost fish and reptiles [@Spoelhof2020a]. In animal groups where diploids and polyploids are both present there may be interploidy hybridisation and subsequent introgression, though based on the published literature this is very uncommon, with only three well-studied examples (Table \ref{CH2T1}). In many other cases where taxa with contrasting ploidies mate introgression is limited, as the hybrid derivatives are hybridogenetic taxa which lack recombination. For example, the edible frog *Pelophylax esculentus* is an extremely ecological successful and widespread hybrid species formed between the diploid taxa *P. ridibundus* and *P. lessonae*. *P. esculentus* includes two cytotypes, a diploid and a triploid, with the triploid being formed and maintained by haploid sperm fertilising the unreduced eggs from a diploid hybrid female [@Hoffmanetal2015a]. However, this taxon appears to be in a state of flux, with no documented all-triploid populations, and tetraploids are extremely rare. Opportunities for novel allelic combinations and introgression are limited as the parental genomes rarely recombine.

\newpage{}

| Family          | Hybridising species                                                                             | Location                                    | Direction to                    | Reference                                 |
| --------------- | ----------------------------------------------------------------------------------------------- | ------------------------------------------- | ------------------------------- | ----------------------------------------- |
| **Animals**     |
|                 |
| Bufonidae       | *Bufo turanensis* (2n = 2x  =22) x *Bufo pewzowi* (2n = 4x = 44)^allo^                          | Kyrgyzstan                                  | Diploid                         | [@Stocketal2010a]                         |
| Cyprinidae      | *Squalius alburnoides* (2n = 2x = 50; 3n = 75; 4n = 100) x *S. pyrenaicus* (2n = 2x = 50)^allo^ | Iberia                                      | -                               | [@Alvesetal2001a; @CrespoLopez2007a]      |
| Myobatrachidae  | *Neobatrachus sutor* (2n = 2x = 24) x *N. kunapalari* (2n = 4x = 48)^auto^                      | Australia                                   | Tetraploid                      | [@RN1469]                                 |
| **Plants**      |
|                 |
| Aspleniaceae    | *Asplenium scolopendrium* (2n = 2x = 72) x *A. adiantum-nigrum* (2n = 4x = 144)                 | Britain                                     | -                               | [@RN1007]                                 |
| Cyatheaceae     | *Gymnosphaera denticulata* (2n = 2x = 138) x *G. metteniana* (2n = 4x = 274)^allo^              | China                                       | Tetraploid                      | [@RN1410]                                 |
| Dryopteridaceae | *Polystichum setiferum* (2n = 2x = 82) x *P. aculeatum* (2n = 4x = 164)                         | Britain                                     | -                               | [@RN912]                                  |
| Asteraceae      | *Achillea clypeolata* (2n = 2x = 18) x *A. collina* (2n = 4x = 36)^allo^                        | Bulgaria                                    | Tetraploid                      | [@RN1444]                                 |
| Asteraceae      | *Centaurea pseudophrygia* (2n = 2x = 22) x *C. jacea* (2n = 4x = 44)                            | Czech Republic                              | -                               | [@RN1041]                                 |
| Asteraceae      | *Cirsium carniolicum* ssp. *rufescens* (2n = 2x = 16) x *C. palustre* (2n = 4x = 34)            | France                                      | Tetraploid                      | [@RN1445]                                 |
| Asteraceae      | *Ixeris repens* (2n = 2x = 16) x *I. debilis* (2n = 6x = 48)^auto^                              | Japan                                       | Hexaploid(?)                    | [@RN1446]                                 |
| Asteraceae      | *Packera paupercula* (2n = 4x = 44) x *P. indecora* (2n = 8x = 88)                              | USA; Michigan                               | -                               | (Kowal et al., 2011)                      |
| Asteraceae      | *Senecio madagascariensis* (2n = 2x) x *S. pinnatifolius* (2n = 4x)                             | Australia                                   | -                               | [@RN1440]                                 |
| Asteraceae      | *Senecio squalidus* (2n = 2x = 20) x *S. vulgaris* (2n = 4x = 40)^allo^                         | Britain                                     | Tetraploid; chromosome doubling | [@RN1447; @RN911; @RN315; @RN980; @RN311] |
| Betulaceae      | *Betula nana* (2n = 2x = 28) x *B. pubescens* (2n = 4x = 56)^allo^                              | Britain                                     | Tetraploid; both(?)             | [@RN1394; @RN1392; @RN1182]               |
| Betulaceae      | *Betula pendula* (2n = 2x = 28) x *B. pubescens* (2n = 4x = 56)^allo^                           | Britain                                     | Tetraploid                      | [@RN908]                                  |
| Betulaceae      | *Betula × purpusii* (2n = 5x = 70) x *B. alleghaniensis* (2n = 6x = 84)^allo^                   | Michigan; USA                               | Hexaploid                       | [@RN1448]                                 |
| Brassicaceae    | *Cardamine apennina* (2n =2x = 16) x *C. amporitana* (2n = 4x = 32)                             | Italy                                       | Tetraploid                      | [@RN1408]                                 |
| Brassicaceae    | *Cardamine × insueta* (2n = 3x = 24) x *C. pratensis* (2n = 4x = 32)                            | Switzerland                                 | -                               | [@RN1235]                                 |
| Brassicaceae    | *Cochlearia officinalis* (2n = 4x = 24) x *C. danica* (2n = 6x = 42)                            | Britain                                     | Tetraploid                      | [@RN1395]                                 |
| Brassicaceae    | *Draba incana* (2n = 4x = 32) x *D. norvegica* (2n = 6x = 48)^allo^                             | Scandinavia                                 | -                               | [@RN1449]                                 |
| Brassicaceae    | *Draba nivalis* (2n = 2x = 16) x *D. daurica* (2n = 8x = 64)                                    | Scandinavia                                 | -                               | [@RN1449]                                 |
| Brassicaceae    | *Draba arctica* (2n = 10x = 80) x *D. corymbosa* (2n = 16x = 128)^allo^                         | Scandinavia                                 | -                               | [@RN1449]                                 |
| Brassicaceae    | *Rorippa austraica* (2n = 2x = 16) x *R. sylvestris* (2n = 4x/6x = 32/48)                       | Germany                                     | Both                            | [@RN1393]; see also @RN1450               |
| Fabaceae        | *Lotus stepposus* (2n = 2x = 12) x *L. × ucrainicus* (2n = 4x =24)^allo^                        | Ukraine, Turkmenistan, Kazakhstan, Mongolia | -                               | [@RN1451]                                 |
| Liliaceae       | *Erythronium mesochoreum* (2n = 2x = 22) x *E. albidum* (2n = 4x = 44)                          | Nebraska; USA                               | -                               | [@RN1452]                                 |
| Orchidaceae     | *Dactylorhiza fuchsii* (2n = 2x = 40) x *D. praetermissa* (2n = 4x = 80)^allo^                  | Belgium                                     | -                               | [@RN1067]                                 |
| Orchidaceae     | *Dactylorhiza incarnata* (2n = 2x = 40) x *D. praetermissa* (2n = 4x = 80)^allo^                | Belgium                                     | -                               | [@RN1067; @RN1453]                        |
| Orchidaceae     | *Dactylorhiza incarnata* subsp. *cruenta* (2n = 2x = 40) x *D. lapponica* (2n = 4x = 80)^allo^  | Norway                                      | Tetraploid                      | [@RN1053]                                 |
| Orchidaceae     | *Dactylorhiza incarnata* (2n = 2x = 40) x *D. traunsteineri* (2n = 4x = 80)^allo^               | Sweden                                      | Tetraploid                      | [@RN1454]; see also @RN1441               |
| Orchidaceae     | *Dactylorhiza fuchsii* (2n = 2x = 40) x *D. maculata* (2n = 4x = 80)^auto^                      | Europe to Caucasus                          | -                               | [@RN1455]                                 |
| Orchidaceae     | *Epidendrum fulgens* (2n = 2x = 24) x *E. puniceoluteum* (2n = 4x = 52)                         | Brazil                                      | Tetraploid                      | [@RN905]                                  |
| Orobanchaceae   | *Euphrasia anglica* (2n = 2x = 22) x *E. micrantha* (2n = 4x = 44)^allo^                        | Britain                                     | Diploid(?)                      | [@RN423; @RN383]                          |
| Phrymaceae      | *Mimulus guttatus* (2n = 2x = 28) x *M. luteus* (2n = 4x = 60-2)^allo^                          | Britain                                     | Chromosome doubling             | [@RN1383]                                 |
| Plantaginaceae  | *Callitriche cophocarpa* (2n = 2x = 10) x *C. platycarpa* (2n = 4x = 20)^allo^                  | Europe                                      | -                               | [@RN1456]                                 |
| Poaceae         | *Vulpia fasciculata* (2n = 4x = 28) x *Festuca rubra* (2n = 6x = 42)                            | Britain                                     | Hexaploid(?)                    | [@RN1385]                                 |
| Polygalaceae    | *Polygala calcarea* (2n = 2x = 34) x *P. vulgaris* (2n = 4x = 68)                               | Britain                                     | Tetraploid                      | [@RN1457]                                 |
| Polygonaceae    | *Fallopica sachaliensis* (2n = 4x = 44) x *F. japonica* var *japonica* (2n = 8x = 88)           | Britain                                     | -                               | [@RN1396]; see also @RN1458 and @RN1459   |
| Polygonaceae    | *Rumex obtusifolius* (2n = 4x = 40) x *R. aquaticus* (2n = 20x = 200)                           | Britain                                     | 20-ploid                        | [@RN1460]                                 |
| Primulaceae     | *Dodecatheon frenchii* (2n = 2x = 44) x *D. meadia* (2n = 4x = 88)                              | Illinois; USA                               | Tetraploid                      | [@RN1084]                                 |
| Rannunculaceae  | *Aconitum variegatum* (2n = 2x = 16) x *A. firmum* (2n = 4x = 32)^allo^                         | Europe                                      | Diploid?                        | [@RN1169]                                 |
| Rannunculaceae  | *Ficaria calthifolia* (2n = 2x = 16) x *F. verna* subsp. *verna* (2n = 4x = 32)                 | Europe                                      | -                               | [@RN1167]                                 |
| Rosaceae        | *Rosa rugosa* (2n = 2x = 14) x *R. mollis* (2n = 4x = 28)                                       | Europe                                      | Tetraploid                      | [@RN1461]                                 |
| Violaceae       | *Viola reichenbachiana* (2n = 2x = 20) x *V. riviniana* (2n = 4x = 40)^allo^                    | Germany                                     | -                               | [@RN1462]; see also @RN1074               |

Table: Studies reporting cross-ploidy hybrids based on cytological and/or molecular genetic analyses. Details are provided of plant family, hybridising species, broad geographic locality, and the direction of introgression (if known). Superscripts indicate whether the polyploids are allopolyploid (^allo^) or autopolyploid (^auto^). \label{CH2T1}


## Biology of cross-ploidy hybrids: general features 

Cross-ploidy hybrids can arise in a variety of situations. Many, but not all, examples occur in contact zones between parental species, where hybrid zones and hybrid swarms may form. Some of these hybrid zones have shifted over time [e.g. *Betula*, @RN1394], or are mosaic in structure [@RN1167]. In addition, there are notable differences in genetic structure between contact zones, with some comprising a swarm of F1, F2 and backcrossed hybrids [@RN1395], indicating low genetic divergence between parental species [@RN1294], while others contain only a few early generation hybrids, suggesting that parental species are more distantly related, and show higher levels of pre and post-zygotic isolation [@RN1041]. Moreover, the direction of introgression is overwhelmingly towards the higher ploidy parent (21 out of 26 studies in Table \ref{CH2T1} that reported directionality). This is unsurprising as the union of an unreduced 2n = 2x gamete of a diploid and a reduced n = 2x gamete of a tetraploid provides a direct pathway for introgression in this direction, whereas the alternative direction is a two-step process via the triploid bridge [@RN1020; @RN1380]. As such, only two plant studies and one animal study report the opposite scenario [*Aconitum* and *Euphrasia*, *Neobatrachus*; @RN1169; @RN423; @RN1469], and a further two studies report bidirectional introgression [in *Betula* and *Rorippa*, @RN1392; @RN1393]. However, other factors may still pose limits for introgression in the direction of the higher ploidy parent. Polyploids evolve meiotic stability to ensure reliable segregation of additional chromosomes at meiosis, with loci underlying tetraploid meiotic stability shown to be under selection in natural populations of autotetraploid *Arabidopsis arenosa* [@Hollisteretal2012a]. Cytogenetic evidence in *Arabidopsis* suggests introgression from diploids to tetraploids may introduce genetic variants that disrupt regular meiosis in tetraploids [@RN1433].  

Hybrids may also occur in the absence of one or both parents, normally where greater lifespans allow persistence long after hybrid formation [@RN1396; @RN1120]. Where cross-ploidy hybrids are present without their parents, they may represent stable lineages that survive through asexual reproduction (e.g. vegetative reproduction or apomixis), and are therefore different to some ephemeral forms present in hybrid zones. On occasion, cross-ploidy hybridisation has led to recent speciation (<200 years). This has occurred in the plant genera *Senecio* [@RN640; @RN311] and *Mimulus* [@RN1383]. These hybrids are also notable in the context of the British Isles, as they involve alien species as either one, or both parental species. Further examples where cross-ploidy hybridisation involves aliens species are in *Rosa rugosa* and both parental species of *Fallopia* (Table \ref{CH2T1}). Human mediated translocations of species therefore continue to have a profound effect on hybridisation. Older hybrid species (10,000+ years) have also originated in a similar way to *Senecio* and *Mimulus* hybrid species, with this inferred either through morphology and cytogenetic analysis, or through sequence analysis showing ‘ghost’ subgenomes of allopolyploid species [e.g. *Euphrasia*, *Packera*, @RN423; @RN1397]. 

A key determinant of genetic variation in cross-ploidy hybrids will be whether the polyploid parent(s) are auto or allopolyploids. In allotetraploid parents characterised by disomic inheritance, preferential chromosome pairing between the most similar, homeologous subgenomes, may lead to a subset of polyploid variation introgressing. In contrast, in autotetraploids with tetrasomic inheritance, free recombination between chromosomes may allow any region of the tetraploid to introgress. In our literature survey, we reported what kind of polyploid the higher ploidy parent in the cross was. 20 out of the 23 studies which contained information on polyploid type reported allopolyploids. While allopolyploids garner more research interest than autopolyploids in studies of hybridisation [@RN1473], the higher number of studies reporting allopolyploids may be biologically significant. For example, chromosome pairing of an allotetraploid subgenome more related to the diploid parent could lead to higher probabilities of successful hybridisation than in diploid-autotetraploid hybridisation, where chromosome pairing would be disrupted.

In addition to interploidy hybridisation between species, much early work, both theoretical and empirical, has explored crosses within mixed-ploidy species complexes [@RN1403; @RN1406; @RN1407]. The outcomes of crosses within (diploid x autopolyploid) or between species (diploid x autopolyploid/allopolyploid; Box 3) are similar in many cases; with triploid hybrids still formed [@RN1405; @RN1067], unreduced gametes remaining an important driver of hybridisation [@RN1408; @RN1380], and the direction of introgression is usually towards the higher ploidy parent [Table \ref{CH2T1}; @RN1356; @RN905]. On the other hand, between species hybridisation can lead to higher levels of genetic variation through fixed heterozygosity in hybrids, and backcrossing to parental species, resulting in higher fitness [@RN1381]. In addition, the higher the divergence between species, the higher the likelihood of whole genome duplication post hybridisation, and therefore the generation of novel polyploid species [@RN1161].

\newpage{}

---------------------------------------------------------------------------------
Box 3: Outcomes of cross-ploidy hybridisation
---------------------------------------------------------------------------------
The evolutionary outcomes once a hybrid has been generated are diverse and depend upon factors relating to hybrid creation frequency, population sizes of parental species, niche separation of hybrid and parental species [@RN1372], the direction of introgression [@RN1020], hybrid fitness [@RN1373], and hybrid fertility [@RN1115]. Taken together, these myriad barriers pose problems not only to the formation, but also to the establishment of cross-ploidy hybrid lineages.

After a cross-ploidy hybrid has formed, three outcomes may occur. The hybrid individual or population may either die before reaching maturity or go extinct, act as a conduit to gene flow between ploidy levels, or persist and establish to form a new hybrid entity or species. Firstly, extinction of the hybrid is highly likely if it is formed at low frequencies and parental species are rare [i.e. low propagule pressure; @RN1372]. The growth and development of the hybrid can be affected by bringing together incompatible parental allelic combinations, causing the hybrid to be unfit [e.g. hybrid necrosis; @RN1371]. Ultimately, fertility of an F1 hybrid will determine its persistence in a population. Triploid F1 hybrids that overcome triploid block often display very low fertility (Figure \ref{CH2BF2} panels a and b) due to irregularities at meiosis which form aneuploid gametes [@RN1298]. Tetraploid hybrids formed from unreduced gametes (Figure \ref{CH2BF1} panel c) have higher fertility than triploids [@RN1115]; however there is no evidence to suggest that newly formed allotetraploids have higher fertility than autotetraploids, which may be expected if pairing behaviour is more regular in allotetraploids [@RN1381].

Given that an F1 hybrid can produce (even rare) fertile gametes, low levels of outcrossing can promote gene flow between ploidy levels through backcrossing with parental species. For a triploid F1 hybrid, there are two pathways to generate a backcross of equivalent ploidy to one of the parental species. Firstly, the triploid F1 may produce reduced pollen which combines with reduced pollen from the diploid male parent (Figure \ref{CH2BF2} panel a) which has been hypothesised to occur in *Euphrasia* and *Aconitum* [@RN423; @RN1169]. Secondly, the triploid F1 hybrid can produce unreduced gametes that can either combine with reduced gametes from the tetraploid parent or unreduced gametes from the diploid parent [Figure \ref{CH2BF2} panel b; e.g. *Senecio eboracensis*; @RN640]. Tetraploids therefore are much more readily produced, as in addition to the two pathways mentioned, tetraploids can be produced in a single generation following cross-ploidy hybridisation (Figure \ref{CH2BF1} panel c). The bias towards tetraploid production has been known since Stebbins in the 1950s [@RN1356] and is the reason why introgression in the direction of the tetraploid is more common [@RN1380].

For persistence of a hybrid lineage to occur, reproductive isolation between the newly formed hybrid and the parental progenitors is paramount. Unlike cases of polyploid hybrid speciation where the hybrid is of differing ploidy level to both parents, backcrossed F1 hybrids derived from cross-ploidy hybridisation will match one parental ploidy and therefore lack the strong reproductive barrier that polyploidy confers. In this case, other factors contribute to reproductive isolation, including ecological selection, niche differentiation, selfing, and chromosomomal or genetic sterility barriers [@RN899; @RN989; @RN1382]. Lastly, reproductive isolation of a cross-ploidy hybrid can occur by the doubling of the triploid F1 chromosome complement to produce a fertile hexaploid that is isolated by ploidy level from the parental species. This scenario has been recorded twice in recent history and has given rise to two neoallohexaploid species, *Senecio cambrensis* [@RN311] and *Mimulus peregrinus* [@RN1383].

---------------------------------------------------------------------------------

![Potential outcomes of a triploid F1 backcrossing to the parental species. In both panels, the schematic follows that of Figure \ref{CH2BF1} panel a. Interrupted lines indicate backcrosses to parental species. In panel a the triploid F1 hybrid produces reduced gametes that combine with reduced gametes from the diploid male parent. In panel b there are two pathways to produce a tetraploid F1 backcross: firstly the unreduced gametes from the triploid F1 can combine with reduced gametes from the female tetraploid parent, secondly the unreduced gametes from the triploid F1 can combine with unreduced gametes from the diploid male parent. Figure generated with graphviz [@RN1401].\label{CH2BF2}](./PDFs/Cross_Ploidy_F2.pdf){ width=80% }

It has been proposed that within polyploid complexes a widespread tetraploid could acquire genes via unilateral introgression from ecogeographicaly isolated diploid taxa occurring sympatrically with it in different parts of its range [@RN1356]. In this way, several different forms of a tetraploid might originate, with each one bearing a close resemblance to the local diploid it hybridised with. Based on cytotaxonomic evidence, @RN1356 & @RN1020 suggested this has occurred in numerous polyploid complexes of a number of plant genera, including *Dactylis*, *Knautia*, *Grindelia*, *Phacelia* and *Campanula*. Recently, genomic evidence has been obtained to provide support for Stebbins’ proposal from work conducted on a polyploid complex comprising diploid and tetraploid forms of *Arabidopsis arenosa* in Europe [@Arnoldetal2015a]. Genomic analysis indicates that autotetraploid *A. arenosa* arose once and then split into five major lineages as it spread into different parts of Central Europe [@Arnoldetal2015a]. For two of the lineages, there is evidence that particular haplotypes, not found in any other tetraploid lineage, are shared with proximal diploid forms of *A. arenosa*, indicating these haplotypes were acquired from the local diploid type and are adaptive [@Arnoldetal2015a]. In addition, one of the five tetraploid lineages is a ruderal form, widely distributed along the railways of Central and Northern Europe. Subsequent analysis indicates that the widespread lowland form of this early flowering and rapid cycling “railroad ecotype” likely originated as a result of introgression of genes from diploid *A. arenosa* occurring on the Baltic Coast of Germany and Poland into local populations of the tetraploid [@RN1380; @RN1468].

## Future perspectives 

While cross-ploidy hybridisation is likely more common than previously thought, particularly in plants, there is still much uncertainty in our understanding of the phenomenon. To better determine the frequency of cross-ploidy hybridisation, we need to broaden the taxonomic scope under study. There is currently a dearth of information on animal examples, even though polyploid incidence can be high in some groups [e.g. insects, decapods, fish, and amphibians; @RN1185]. Further, while we found many angiosperm examples, half were derived from the large families Asteraceae and Orchidaceae. A broader scope will also determine more readily whether there is a phylogenetic signal to the phenomenon, and which attributes, from ecological to genetic factors, facilitate cross-ploidy hybridisation and introgression. More detailed mechanistic research across a wide variety of taxa will reveal the underlying genomic variants that allow chromosomes to pair in newly formed polyploid hybrids [@RN1433], which is important in establishment and persistence of hybrids. Most research on cross-ploidy hybridisation so far has focused on either contact zones or cryptic introgression; studying stabilised hybrids outside of these situations will provide a more detailed picture of how these lineages persist, and under which conditions [e.g. see @RN256 and references within]. Next generation sequencing promises to reveal cross-ploidy hybridisation more easily [@RN1410], quantify the directionality of introgression accurately [@RN908], and determine parental genomic contributions to cross-ploidy hybrids [@RN1411]. The latter point is particularly important, as hybrids may be introgressed at only a few loci in the genome. Detecting these few loci requires a good polyploid genome assembly, preferably with phase information, and new sequencing methods and software are beginning to address these problems [@RN1412]. Specifically, long-read Oxford Nanopore Technologies and Pacific BioSciences sequencing, as well as Hi-C and BioNano for scaffolding will produce highly improved, contiguous genome assemblies, and population long-read sequencing will be able to detect fine level introgression more easily. In addition, sequencing of diploid relatives, and haploid tissue in ferns will allow us to distinguish between subgenomes and work out phase. Given the extensive ploidy variation throughout plants and animals, and the high degree to hybridisation is detected in these groups, cross-ploidy hybridisation may be more frequent and important in plant and animal evolution than is currently thought.
