\newpage{}

# General introduction

## Taxonomic complexity in plants 

 Much of biology is built upon the idea that the diversity we see in nature can be categorised into discrete units, which at its heart is the concept of a species [@RN1474]. One of the most important species concepts is the biological species concept (BSC), where species are recognised as a community of interfertile individuals [@RN1475]. The BSC fits many organisms, and is the foundation of evolutionary research, especially in plants and animals [@Butlinetal2020]. Although plants contain some notoriously complex groups, most taxonomic species and phenotypic clusters represent reproductively isolated lineages [70%+; @Riesebergetal2006a]. The concept of species in plants is therefore biologically meaningful and useful in the majority, with phenomena such as asexuality and polyploidy at the forefront of species delimitation issues.

Ever since this task of discriminating and categorising species has been embarked upon, difficulties have emerged as to how to delimit certain species, mainly because the processes that give rise to species are in constant motion [@RN1477]. Arguably some of the most difficult groups to classify are collectively placed under the umbrella term of ‘taxonomically complex groups’ [TCGs; @RN269]. A TCG is here defined as one where it is difficult to categorise the biodiversity present due to underlying processes which blur species boundaries. TCGs are problematic, not only in terms of classification but also as to how to best conserve them [@RN788; @RN1478]. TCGs are present across the tree of life, from fungi [@RN1479], to fish [@RN1480], and arthropods [@RN1481]. TCGs are remarkably frequent in plants however, and particularly common in certain families such as Poaceae [@RN1483], Rosaceae [@RN940], and Asteraceae [@RN941]. TCGs can be the result of plant populations that no longer undergo sexual, random mating [@RN407], but other TCGs are the result of phenotypic plasticity and recent speciation blurring species boundaries [@RN1482; @RN1326]. 

Factors commonly contributing to TCGs include selfing, apomixis (or agamospermy), hybridisation, and polyploidy. Selfing and apomixis disturb random mating and restrict gene flow between populations, and often interact with hybridisation and polyploidy [@RN407]. For example, apomixis coupled with rare hybridisation can produce arrays of microspecies, characteristically seen in genera such as *Rubus* (330+ microspecies in Britain), *Hieracium* (400+) and *Taraxacum* [240+; @RN1309]. Mating system shifts from outcrossing to selfing or apomixis, drive rapid reproductive isolation from progenitor lineages through founder effects, genetic drift, and selection of advantageous recessive alleles [@RN407]. This leads to strong population structure, characterised by many and varied subtle morphological changes between populations. Similar phenomena result from ploidy level variation [@RN1484; @RN1485], however hybridisation can assist in moving genetic material between populations, which can produce yet more morphological variation [@RN1359]. Species concepts usually break down in TCGs, as the evolutionary processes creating new variation overwhelm any stable pattern of species, and the relationships between species become subtle, finely divided, and overlap.

## The role of hybridisation in taxonomic complexity

Hybridisation, defined here as the mating between different species, is an important factor driving taxonomic complexity [@Stebbins1959a; @RN269; @RN1486]. The role of hybridisation is not simple however, as many different outcomes are possible. Hybridisation can be destructive, where rare species may lose their genetic integrity, resulting in populations of entirely hybrid genotypes, and can eventually lead to the extinction of the rarer species [@RN935; @RN1487]. On the other hand hybridisation can be creative, by allowing adaptive traits to move between species [@RN980]. Hybridisation can also lead to introgression, where there is the incorporation of genetic material from one species in the genetic background of another [@RN404]. In the extreme, new species can be formed in a process known as hybrid speciation [@RN991]. There are two main pathways to speciation involving hybridisation: polyploid hybrid speciation where the hybrid species has duplicated its chromosome complement [allopolyploidy; @RN1021], and homoploid hybrid speciation where the parental species and the hybrid remain at the same ploidy level [@RN989]. Many TCGs involve hybridisation which blurs species boundaries, coupled with processes that may allow the hybrid derivatives to persist [@RN269]. Three categories in which TCGs can be placed include agamic complexes, in which hybridisation is combined with a mode of asexual reproduction (e.g. apomixis) to propagate lineages [@RN1488], polyploid complexes, where the hybrid derivatives are sexual polyploids [@RN1489], and homogamic complexes [or homoploid complexes, e.g. *Helianthus* @RN927], where hybrid derivatives are mainly diploid and isolated from parental progenitors ecologically. 

Some TCGs defy these three broad categories, by combining properties of different species complexes. A good example of this is the genus *Sorbus* in the British Isles, where there are 45 taxa, plus seven more which have been introduced [@NelsonJonesetal2002; @RN1490]. In *Sorbus*, hybridisation, polyploidy, and apomixis have interacted to form numerous endemic species in England, Scotland, and Wales [@RN1491]. In England alone, this process happens because there are few sexual diploid species (*S. torminalis*, *S. aria*, and *S. aucuparia*, but not *S. domestica*) which hybridise with apomictic polyploid derivatives of the species *S. aria* [at least 20 taxa; @RN1490; @RN1492]. Apomictic *Sorbus* species require pollen to achieve successful asexual reproduction, however at low frequencies the pollen can fertilise the maternal embryo and this leads to a stable, new, polyploid, apomictic taxon [@RN1491]. In all, more than 31 apomictic species have arisen in this way [@RN1007]. The apomictic condition ensures that any new hybrid genotype is frozen in stasis, and can lead to complex reticulate evolutionary histories, which is seen in many other apomictic plant systems [@RN1493; @RN1054]. Not only is hybridisation important in generating biological diversity and complexity, it is also a common phenomenon both geographically, and phylogenetically [@RN1286]. The clear and widespread abundance of hybridisation means it has had, and continues to have, a profound effect on the evolution of plants – especially in conjunction with selfing, apomixis, and polyploidy.

## The role of polyploidy in taxonomic complexity

Polyploidy is the condition where a cell contains more than two sets of chromosomes as a result of whole genome duplication (WGD), and is featured in almost all TCGs in plants [@Brystingetal2007; @RN269]. The two major routes to polyploidy are either through WGD of a single species chromosome complement, known as autopolyploidy, or through hybridisation between two species followed by WGD, known as allopolyploidy [Figure \ref{CH1F1}; however, there are other mechanisms; see @RN918]. As polyploid individuals tend to be larger, more vigorous, and quicker growing, it is no surprise that many crop species are polyploid [e.g. wheat, rice, potato, maize @RN1520]. Although the majority of extant plant species are diploid [~67%, @RN1189], extensive variability in ploidy levels exist across flowering plants, at all taxonomic levels [@RN1344; @RN1343]. Most if not all plant species have experienced historical WGD a number of times [@RN1521]. Over evolutionary time the process of diploidisation reduces the size of these previously polyploid genomes through loss of repetitive sequence, chromosomal rearrangements, and reductions in chromosome number [@RN1494]. Both the spatial and phylogenetic distribution of ploidy variation are unlikely to be uniform due to climatic and clade specific effects on unreduced gamete formation, which is the main driver in the creation of polyploid organisms [@RN1171; @RN1400; @RN1189]. 

![A simplified diagram of the formation of tetraploids from diploid progenitors. a) the formation of an autotetraploid from two diploid related individuals (red and pink) and b) the formation of an allotetraploid from two divergent diploid species (red and blue). Here, triploid intermediates can be formed through the combination of unreduced and reduced gametes. These triploids can then either self-fertilise, or backcross with parental species to produce tetraploids. Alternatively the union of unreduced gametes from both parents can produce a tetraploid in one step. Note: the distinction between auto and allopolyploids are blurred as the parental species may only be partially differentiated [@RN1473], and there are other rarer mechanisms to produce auto and allopolyploids involving genome doubling.\label{CH1F1}](./PDFs/Auto_allo2.pdf){ width=80% }

\newpage{}

Polyploidy is a contributing factor to taxonomic complexity, with recurrent polyploid formation followed by hybridisation within and between ploidy levels being the main driver of some TCGs [@RN1495]. Further, there is evidence to suggest that polyploidy facilitates hybridisation as ploidy level increases, i.e. tetraploid x tetraploid crosses are more successful than diploid x diploid crosses [@RN1469; @RN1468]. Shifts in mating system from outcrossing to selfing or apomixis are also correlated with polyploid formation, due to the breakdown of self-incompatibility, selection against minority cytotype exclusion, and the capability of polyploids to alleviate inbreeding depression [@RN347]. Recurrent polyploidisation has been shown to generate taxa of ever higher ploidy levels, some of which may persist to form new species, or backcross with parental species to form a complex reticulate group [@RN1496]. Polyploid type (auto or allopolyploid) is important in determining their role in taxonomic complexity. Autopolyploids and allopolyploids are both frequent in nature; current estimates for both kinds of polyploids are around 10% [@RN1345]. Autopolyploids form complexes within species where ecology can shape the distribution of cytotypes [e.g. @RN1497], however these cytotypes are rarely considered species (Ramsey and Ramsey, 2014). Allopolyploids contribute to taxonomic complexity by combining two diverged genotypes to form a usually distinct and isolated taxon from the parents [@RN1498]. Both kinds of polyploid are present within some TCGs [e.g. *Sorbus* and *Cochlearia*; @RN1491; @RN1270].

Unusual cytogenetic features in some plant groups have generated TCGs that have defied classification for over a hundred years. Two examples are the dog roses in *Rosa* section Caninae and the evening primroses in *Oenothera* section Oenothera [@RN1398; @RN1499]. The dog roses present an unusual breeding system where the female parent contributes 3-5 copies of the genome and the male parent contributes only one [section Caninae species are usually pentaploid; @RN1500]. This results in the hybrids containing a genome which is mostly maternal, and has the added complication that reciprocal crosses entirely change the genomic constitution of the hybrid. Species are difficult to tell apart when this breeding system is combined with hybridisation between rose species, as hybrids are fertile and readily backcross, forming swarms of individuals that may be impossible to identify morphologically [@RN1007]. Another example of strange cytogenetic behaviour, but at the diploid level, are the evening primroses. Here, the species exhibit a breeding system where translocation hybrids and balanced lethals produce chains of chromosomes which are inherited as single units [@RN1499]. While evening primroses breed true when selfing, different combinations of chromosome chains generated through hybridisation can produce completely new morphological taxa [@RN1501]. This introduces taxonomic complexity as, like the roses, when two species come into contact they form hybrid swarms of intermediate genotypes, each of which would breed true in isolation [@RN1007].

## Novel features of TCGs; parasitism and plasticity

Approximately 1% of all angiosperm species are parasitic, with some genera being particularly speciose such as *Pedicularis* (c.a. 650 sp.), *Euphrasia* (c.a. 260 sp.), and *Thesium* [c.a. 300 sp. @RN1434; @RN1502; @RN1140]. Parasitic plants are defined by the formation of a structure called the haustorium (Figure \ref{CH1F2}), which is used to extract water and soluble nutrients from the host plant [@RN1140]. The haustorium can attach either to roots [e.g. *Orobanche*; @RN1503] or shoots [e.g. *Viscum*; @RN1504], or rarely the haustorium is present inside the host plant itself [e.g. endoparasitic *Pilostyles*; @RN1506]. There are two types of parasitic plants – hemiparasites which retain photosynthetic competency, and holoparasites which are devoid of chlorophyll and entirely dependent on host plants [@RN842]. Host species range and identity varies widely between parasitic plants, and some generalists can parasitise more than 100 host plant species [e.g. *Amyema miquelii*; @RN1505], while others specialise on a single or few host plants [e.g. *Epifagus virginiana* on *Fagus grandiflora*; @RN1148].

![A variety of parasitic plants, showing haustorial connections. a) *Cuscuta europaea* (dodder), showing flowers and twining red stems. b) *Erianthemum ngamicum* showing its large terminal haustorium. c) *Orobanche hederae* haustorial connection to the host root (light) d) *Hydnora visseri* forming multiple haustoria on host root (light). e) *Cassytha pubescens* twining around host forming many haustorial connections. f) *Viscum album* showing self-parasitism where two younger *V. album* individuals have established on an internode of an older specimen. g) Terminal (asterisk) and lateral (arrows) haustoria of *Plicosepalus kalachariensis*. h) flowers of *Agelanthus gracilis*. Figure and text adapted from @RN842.\label{CH1F2}](./PDFs/haustoria.png){ width=80% }

How different host species may influence parasitic plant phenotypes however, has been little explored in relation to taxonomic complexity. Yet it is well known that different host species can dramatically impact the growth, development, and evolution of parasitic plant individuals [@RN1152; @RN963]. There are two main ways in which the parasitic lifestyle may contribute to taxonomic complexity. Firstly, parasitic plant species may show phenotypic plasticity when utilising different host species, which may be substantial enough to confuse species identification [@RN949]. For example, a suite of traits used in species discrimination in the Orobanchaceae, including corolla length, node to first flower, and plant height, all vary in relation to host quality [@RN1152; @RN379]. Secondly, differential host use can drive the evolution of cryptic taxa, which has been seen in the plant genus *Orobanche*, which specialise on different host species [@RN386].

## The genus *Euphrasia*

The genus *Euphrasia* (eyebrights) are a large group of 260, mainly annual but sometimes perennial, hemiparasitic plant species in the Orobanchaceae [@RN1434; @RN424]. A study of global *Euphrasia* species has established its bipolar distribution, and estimated the age of the genus to be around 20-30 Mya [@RN391]. *Euphrasia* species are found in Chile, Australia and New Zealand, parts of South East Asia, and widely across the northern hemisphere. It is here, in the northern hemisphere that *Euphrasia* species are most diverse and where the genus is a notorious TCG [@RN424]. Within Europe, Britain and Ireland contain the highest concentration of species, where 21 have been described [@RN1156]. The number of described species in Britain may be an exaggeration however due to over-splitting. Here, *Euphrasia* is widespread and occupies a range of habitats, from coastal scrub to heather moorland, and damp grassland to mountain tops [@RN1156]. While the split between diploid species and tetraploid species is unequivocal, within each ploidy level there is much uncertainty in species limits due to rapid recent divergence [@RN1326]. Within tetraploids, some species groups are more distinct that others, for example *E. micrantha* and *E. scottica* form a natural group, as do the widespread outcrossing species *E. arctica* and *E. nemorosa* [@RN383]. Omitting these two species groups, it leaves a nebulous pool of widespread and more localised selfing species that may be genetically partitioned more by geography than by species limits. Within the diploid species, there is evidence to suggest that the endemic diploid species *E. vigursii* and *E. rivularis* are distinct from the other diploids [*E. anglica* and *E. rostkoviana*; @RN383].

The taxonomic complexity of *Euphrasia* in Britain is reflected in the unstable taxonomy over the past hundred years, in the great diversity of morphologies found in the field, and in the complex genetic structure of the species [@RN1507; @RN1156]. The main drivers of this diversity are recent postglacial divergence, the tendency to self-fertilise, and rampant hybridisation – 71 hybrid combinations have been reported to date [Figure \ref{CH1F3}; @RN1326; @RN1007; @RN1156]. At least five species of the British species of *Euphrasia* have a putative hybrid origin, with two species having arisen from diploid-tetraploid hybridisation events [@RN423; @RN1508]. The evidence of cross-ploidy hybridisation began with early cytological work on a triploid hybrid which was intermediate between *E. micrantha* and *E. anglica* (= *E. vigursii*) in morphology [@RN423]. Further evidence was found by @RN383 when tetraploid specific AFLP bands were seen in diploid samples. Lastly, @RN1413 in a genomic analysis of British *Euphrasia*, found some evidence for gene flow between *E. arctica* and the sampled diploid species. These cross-ploidy hybrid species are of particular interest, as they combine three important features of TCGs – hybridisation, polyploidy, and mating system variability.

![A matrix showing interspecific crossing barriers in British *Euphrasia* species. Grey squares indicate intraspecific crosses, pink squares show crosses not known to produce hybrids in the wild, and green squares show those crosses to have produced hybrids in the wild. Diploid species are emphasised in bold type. Data taken from @RN1156.\label{CH1F3}](./PDFs/matrix_plot.pdf){ width=80% }

*Euphrasia* in Britain and Ireland have two different ploidy levels, with diploids (2n = 2x = 22) that have a southern distribution and tetraploids (2n = 4x = 44) more predominant in the north [@RN424]. The tetraploid species are allotetraploids, containing one subgenome which is closely related to extant diploids [0.2% divergent; @RN1413]. This low divergence has led to the hypothesis that pairing can occur between the chromosomes of diploid species and the diploid-like subgenomes of tetraploids, which could explain diploid-tetraploid hybridisation in the genus [see Figure \ref{CH1F4}; @RN423]. The divergence between diploids and tetraploids is around 5% based on both ITS sequencing, and genome wide data, which corresponds to a split time of around 8 Mya [@RN1326; @RN1413]. High divergence between diploids and tetraploids points to ploidy being an effective barrier to gene exchange, however there is mounting evidence that gene flow between ploidy levels is present, but rare [@RN383; @RN1413]. Hand crosses have generally failed to produce diploid-tetraploid hybrids, and only a single triploid hybrid has been found in the field [@RN422; @RN423]. Controlled crosses may yet yield useful insights into the biology of diploid-tetraploid hybrids in *Euphrasia*.

![Schematic of diploid-tetraploid hybridisation in *Euphrasia*. Here, a diploid species hybridises with a tetraploid species to form a triploid intermediate. Note that pink and red colours indicate the low [0.2%; @RN1413] divergence between diploid (pink) and diploid-like (red) tetraploid subgenomes. This triploid F1 can then backcross to the diploid parent through rare haploid gamete segregation to produce a backcrossed F2 individual. Note the striped chromosomes in the F2 backcross indicate recombination between the diploid (pink) and the diploid-like (red) subgenome of the tetraploid. It is thought this process has given rise to the hybrid species *Euphrasia vigursii* and *E. rivularis* [@RN423; @RN1508].\label{CH1F4}](./PDFs/HybridspEuphrasia.pdf){ width=60% }

The mating system in British *Euphrasia* is highly variable, and ranges from outcrossing to highly selfing (note apomixis has not been found in the genus). Outcrossing rate is correlated with flower size in *Euphrasia* [r = -0.89; @RN375], with smaller flowered species more likely to self than larger flowered species with showy corollas. For example, the tetraploid mountain specialist *E. cambrica* which is endemic to Wales has a corolla length of 4mm, while the diploid *E. montana* of wet grasslands has a corolla length of up to 12mm [and can be larger in cultivation; Figure \ref{CH1F5}; @RN1156]. Mating system may also impact the directionality of introgression in some cases, for example between tetraploid *E. micrantha* and diploid *E. anglica* which are the putative parental species for the diploid hybrid species *E. vigursii*. *E. anglica* has large flowers and is mainly outcrossing, therefore it is likely to be the pollen donor as its flowers are visited first with greater probability [@RN930]. Therefore the diploid species pollen will be more competitive on the stigma of the tetraploid *E. micrantha* [@RN1022]. As pollen fitness is low in the triploid F1 hybrid, selfing is unlikely to occur and the diploid *E. anglica* is likely to fertilise the F1 [@RN1509]. If this happens over many generations, introgression will occur from the tetraploid to the diploid species (as shown in Figure \ref{CH1F4}).

![corolla size extremes in British *Euphrasia* species. a) shows diploid *E. montana* from the Lake District, England, which has a corolla size of ~12mm. b) shows the Welsh endemic *E. cambrica* from Snowdon, with small flowers ~4mm which rarely open fully [@RN1156]. Photo credit: author.\label{CH1F5}](./PDFs/Corollas_egs2.pdf){ width=80% }

Being generalist parasites, *Euphrasia* can gain benefit from a wide variety of plant species which in turn impact the morphology, growth, and fitness of the parasitic *Euphrasia* plants [@RN947]. Common garden experiments have allowed researchers to investigate these factors in different hemiparasitic plant systems, especially in the Orobanchaceae. For example, it has been shown in *Rhinanthus* that different host species employ different resistance mechanisms to haustorial attack, and this underlies the performance of hemiparasitic *Rhinanthus* [@RN879]. Common garden experiments have also established that host species have strong effects on the biomass and morphology of *Melampyrum* [@RN1152]. As shown from early work by @RN867, *Euphrasia* can easily be brought into cultivation, and their annual life histories (like many other members of the hemiparasitic Orobanchaceae) facilitate experimental work within the timeframe of a PhD.

## The British flora

To create a synthesis of taxonomic complexity, it is useful to be able to frame it in a broad, comparative context. The British flora is an ideal study system, as it contains a manageable number of native plant species (~1,400), but with around 20% of all familial flowering plant diversity [@RN1309]. Alien species increase the total number of species over two-fold, and most of these are very well characterised [@RN287]. The British flora represents the most comprehensively studied flora to date, across a variety of disciplines due to the collaboration between amateur and professional botanists for over a century [@RN1510]. The Botanical Society of Britain and Ireland (BSBI) have played a leading role, holding large databases of plant distributions. The result of this, is that plants can be found and told apart easily in the field, and has led to a huge growth in knowledge of the system. Now for almost all native plants, there is detailed information on plant identification [@RN1309], alien species [@RN287], ecology and life history [@RN1511], chromosome numbers and ploidy level (BSBI Cytology Database), genome sizes (Kew C-value Database), hybridisation [@RN1007], and most recently DNA barcoding (Jones et al., In Review). This wealth of knowledge has been leveraged in parts of this thesis to gain a broader perspective on the topic at hand.

## New methods for investigating taxonomic complexity

In recent years, new methods have emerged that can help us to understand taxonomic complexity, and to resolve major questions. Older marker types such as allozymes, AFLPs, RAPD, and microsatellites had many limitations, including needing large amounts of DNA, problems with homology, detection of few polymorphisms, and lack of reproducibility [@RN1512]. Next generation sequencing (NGS) is one tool to aid resolution of complex taxonomic relationships, for example - genotyping by sequencing (GBS) [@RN1513], restriction associated digestion (RAD) sequencing [@RN1514], target capture [@RN1515], and whole genome sequencing [@RN1517] are becoming common. These datasets can be used to infer relationships across the entire genome, to correlate morphology and genetics to reveal the genes underlying certain traits, and to compare large and diverse sample sets. DNA barcoding can now be deployed at scale across diverse taxa [@Kuzminaetal2017a; @RN1518], which means large scale phylogenetic analyses that incorporate diverse information are now possible. New analytical models now provide a framework for accounting for different sources of variation, and are applicable to use for many different questions across evolutionary biology [e.g. (Generalised) Linear Mixed Models; @RN1519; @RN1157]. These methods are used throughout this thesis to gain novel insights into taxonomic complexity.

## Thesis aims

The main aim of my thesis is to investigate taxonomic complexity in *Euphrasia*, and the role of hybridisation across the British flora, with a particular focus polyploidy, parasitism, and their interactions. It is not currently known what the extent and prevalence of such interactions between hybridisation and plants of different ploidy level are, and is important to explore because of the potentially large impact they can have on plant evolution. In addition, the parasitic habit of some plants and how this affects their phenotype has been little explored and opens up a new avenue of explanations for taxonomic complexity.

The thesis is split into two parts which target different aspects of taxonomic complexity, and draw on broader themes in evolutionary biology. The first part concentrates on the contributions of hybridisation to taxonomic complexity in plants (and to a lesser degree, animals), with a case study in the genus *Euphrasia*. Here, I firstly ask how prevalent hybridisation is between plants that differ in ploidy level, and whether it is a significant evolutionary phenomenon (Chapter 2). The interaction between ploidy level and hybridisation is a poorly explored topic and scattered across the literature. I bring together a comprehensive list of examples, and synthesise current knowledge on this topic. After understanding this global variation, I concentrate on the British flora, and model the probability of hybridisation across the flora (Chapter 3). Significantly, this model includes phylogenetic relationships, and genetic distances based on the first complete DNA barcode dataset across a flora, which previous studies have not been able to comprehensively address [e.g. @RN1286; @RN1285]. Then, I focus on the promiscuous genus *Euphrasia* and ask whether we see evidence of hybridisation between divergent species of different ploidy at a fine spatial scale. Using reduced representation sequencing of genome wide markers (genotyping by sequencing; GBS), I use a combination of classical population genetic tools, genome sequencing, and demographic simulation, to understand the pattern of hybridisation in a *Euphrasia* contact zone (Chapter 4).

In the second part of the thesis, two novel features of taxonomic complexity - phenotypic plasticity and parasitism - are explored in the parasitic plant genus *Euphrasia*. Developing *Euphrasia* as a horticultural system to explore these ideas is discussed first (Chapter 5). I then use common garden experiments to ask how different host species affect the morphology of *Euphrasia* and the ability to discriminate between *Euphrasia* species (Chapter 6). This was investigated using a single species of *Euphrasia* grown across eight different host species and multiple species of *Euphrasia* on a single host, where various morphological traits of *Euphrasia* were quantified. Then I look at the role of host parasite interactions to understand host specialisation in the genus, by growing multiple *Euphrasia* and multiple host species together (Chapter 7). Finally, I synthesise the findings of this thesis in the last chapter, and highlight areas of future research.

