\newpage{}

# The genetic landscape of hybridisation in UK flora

I acknowledge that Natasha De Vere and Laura Jones provided unpublished DNA barcoding data for the British flora, which was used in the analyses presented here.

## Abstract

Hybridisation has a profound impact on the evolution of plants, with consequences including the generation of novel phenotypes [@RN1282], introgression of adaptive alleles [@RN980], and the origin of new species [@RN895], as well as the blurring of species boundaries [@RN1337], and the extinction of rare taxa. While natural hybridisation is common and widespread both geographically and phylogenetically [@RN1284; @RN1014; @RN1286], the ecological factors and species traits that promote or prevent hybridisation in natural plant communities are poorly understood. Many important traits related to hybridisation, such as parental genetic distance and ploidy level differences have not been systematically quantified across diverse species in a flora, while estimates for phylogenetic effects of hybridisation in plant communities are highly variable [@RN1284; @RN1285; @RN1014]. Here, we combine use phylogenetic mixed effect models to combine estimates of hybridisation from extensive field observations [@RN1007] with DNA barcoding data and ecological attributes for over 1,000 native species of flowering plant. Our results quantify the influence of different predictors, and the genetic distances over which hybridisation is most likely. We also quantify the importance of phylogenetic relatedness and parental ploidy differences in shaping the likelihood of hybrid formation. Life history of parental species does not impact hybrid formation despite perennial life history being pervasive in both parental species and hybrids. Although the effect of genetic distance requires careful interpretation as low parental divergence may either allow hybrids to form or be a consequence of genetic homogenisation, taken together with ploidy and phylogenetic effects, genetic factors are key predictors of hybridisation across diverse taxa in the UK flora. 

## Main

Natural hybridisation plays an important role in plant evolution by facilitating adaptation and promoting species survival, generating novel variation, or alternatively in some cases leading to a reduction in fitness and even causing extinction [@RN980; @RN1280; @RN1281; @RN935]. Focused studies on plant evolutionary model systems such as *Helianthus* [@RN1282], *Senecio* [@RN597], and *Tragopogon* [@RN1283] have revealed how hybridisation may contribute to range expansion, invasiveness and phenotypic and genomic novelty. These focused genetic studies have selected species and study systems for their noteworthy hybrid outcomes, and it is currently unclear how these results generalise to natural plant communities and entire floristic assemblages. A relatively small body of work has looked more broadly at hybridisation between diverse species in floras, with the aim of understanding the likelihood of hybridisation in the context of species attributes, ecology, and phylogeny [@RN1014; @RN1284; @RN1285; @RN1286; @RN1287]. A limitation of these studies has been the reliance on phylogenies where the tips represent higher taxonomic units such as genera, families, or orders [@RN1284; @RN1285; @RN1014]. Hybridisation is an outcome of interactions between species, and a resolved species-level phylogeny is required to calculate the genetic distances of hybridising taxa and to accurately estimate the phylogenetic signal of hybridisation. 

DNA barcoding is proving increasingly useful for the genetic characterisation of species assemblages and complex communities [@RN1311; @RN1312]. By amplifying a small and standardised set of target loci, DNA barcoding is inherently scalable to deployment across large sample sets and is well-suited to comparative analyses of diverse taxa. As DNA data is generated at pace and at scale it is becoming feasible to generate comprehensive DNA barcoding datasets for all plant species in a country. This provides the opportunity for the integration with ecological data collected at a national level to test major ecological and evolutionary questions at broad geographic scales.

In this study we characterise the genetic landscape of natural hybridisation across flowering plant species in the UK. The UK flora is an ideal study system for investigating hybridisation as it contains ~1400 species and is therefore manageable for genetic characterisation, but has sufficient diversity to include ~20% of all known angiosperm plant families and numerous hybrid combinations [@RN1007]. There is also a hybrid flora — a unique resource describing all known vascular plant inter-specific hybrids present across the British Isles, of which 616 represent hybridisation between native UK flowering plant species or archaeophytes [introduced pre-1500 or potentially natives @RN1007; @RN1289; @RN1120]. The hybrid flora also summarises their ecology, distribution, cytology and parentage [@RN1007]. We use the Barcode UK dataset (Jones et al. In Review.), a new DNA barcoding resource that includes a three locus DNA barcode of *rbcL*, *matK*, and ITS2 for the native and archaeophyte seed plants of the UK.  We integrate this DNA barcode data with information on hybridisation from the hybrid flora, as well as other species level traits and ecological information. We assess the determinants of hybridisation across the flora in the context of parental species range overlap, genus size, life history (annual vs perennial), ploidy differences, parental genetic distance and phylogeny.

Out of the 6117 possible unique pairwise congeneric combinations between 1100 species suitable for analysis in the UK flora (see Methods), only 7.8% produce hybrids. From 244 genera containing multiple species, 96 contain hybrids, and the 480 recorded hybrids are disproportionately concentrated in just five genera, with 45.8% of hybrids found in *Euphrasia* (n = 62), *Carex* (n = 50), *Rosa* (n = 40), *Epilobium* (n = 35), and *Salix* (n = 33). Some genera have many species that prolifically hybridise (e.g. *Euphrasia* has 4 species that are each parents to 14+ hybrids), while others include widespread species that disproportionately contribute to the number of hybrids (e.g. *Rumex crispus*, involved in 12 hybrid combinations in our data).  We then explored whether the number of hybrids is a simple function of genus size using phylogenetic mixed models. Genus size is predicted to affect hybridisation as larger genera contain more possible pairwise combinations of species, and thus greater opportunity for hybridising with congeners [@RN1288]. Our models show that the probability congeneric species hybridising is independent of genus size (pMCMC = 0.92, Appendix 2 Table 2). Although hybridisation tends to occur in species rich genera, not all species rich genera form hybrids [see also @RN1120]. For example, genera such as *Trifolium* (19 species; usually self-incompatible) and *Alchemilla* (12; all apomictic) are relatively species rich but form no naturally occurring hybrids, whilst *Veronica* (15) and *Galium* (13) form only one naturally occurring hybrid each. This shows that the number of hybrid taxa deviates from a simple model determined by the number of congeneric taxa, with other factors such as phylogenetic relatedness, life history, genetic distance, and ploidy level (explored below) interacting to determine hybridity.

We then investigate hybrid formation in the context of a newly generated 3-locus phylogeny of the British Flora from the Barcode UK data (see Methods). Hybridisation propensity, here defined as the  number of hybrid combinations a species produces weighted by genus size, is highly uneven across the plant phylogeny (Figure \ref{CH3F1}). The phylogenetic signal of hybridisation is high at 0.62 (0.32-0.77 CI – 95% Credible Intervals) meaning that closely related lineages are likely to have similar levels of hybridisation (Appendix 2 Table 3). Inspection of the species level Best Linear Unbiased Predictors (BLUPs; see Methods) from the phylogenetic model shows that the monocot Asparagales clade including orchids (Orchidaceae, Iridaceae, Asparagaceae and Amaryllidaceae) are most likely to hybridise after accounting for other model factors, whilst legumes (Fabaceae) are the least likely (Appendix 2 Figure 2). These results are consistent with previous work; many legumes, such as clovers (*Trifolium*) and related genera (e.g. peas, *Lathyrus*) are known to hybridise very little due to strong between species incompatibilites [@RN1293], while orchids have been shown to hybridise rampantly, even across ploidy levels [@RN1067]. Overall, the pattern of hybridisation is highly heterogeneous across the phylogeny for the British flora, with phylogenetic position a good predictor of hybrid formation.

![Distribution of hybrids across the phylogeny of the British flora. Innermost-ring shows phylogenetic relationships of 1098 British native species from an alignment of ITS2, *matK* and *rbcL*. Phylogenetic reconstructions used maximum likelihood implemented in IQ-TREE. The middle ring (green line) shows species-level hybrid propensity weighted by size of genus. The outer ring shows the probit scale posterior mean of the probability of a particular species hybridising (blue line). The zero line is represented in pale red and positive probit values indicate higher probabilities of hybridisation. The figure is annotated with the five genera with highest probabilities of hybridisation, given variation in model fixed effects, indicated from the sum of the species level posterior means from the phylogenetic model (1. *Epilobium*, 2. *Euphrasia*, 3. *Dactylorhiza*, 4. *Potamogeton*, and 5. *Rumex*), and the 10 largest plant orders around the outside.\label{CH3F1}](./PDFs/Posterior_Means_Circos_UK.pdf){ width=80% }

Previous studies have explored the relationship between life history and hybridisation in plant species [@RN1284; @RN1285; @RN1289; @RN1286; @RN1120]. It has been documented that most hybridising species are perennial, at least in temperate floras [e.g. 97% in the Michigan flora @RN1284, 68% in our data]. Out of all potential congeneric species pairs of the same life history in the British flora however, perennial only parental combinations form proportionally fewer hybrids (7.7%, n = 4725) than annual only parental combinations (15.6%, n = 588). Perennial plant species are thought to participate in hybridisation events more frequently because (a) perennials tend to outcross more than annuals, which tend to be more highly selfing (Morgan, 2001) and (b) perennial plants are longer lived and therefore more gametes are produced over a longer period of time [@RN1286]. Despite the high frequency of perennial parent plant species we find no evidence for a significant effect of life history on the probability of hybrid formation ($\chi^2$ = 4.04, df=2, P=0.13; Appendix 2 Table 4).  Life history may be still be important in the persistence of hybrid lineages [@RN1291], however our results show that the probability of forming a hybrid is independent of parental species life history. 

Increasing overlap in parental species distribution is expected to increase the probability of hybridisation due to the greater opportunity for crossing and therefore hybrid formation. Whilst this is an important variable when considering hybridisation, few studies have critically looked at range overlap with respect to hybridisation in multi-species systems [@RN1285; @RN1120]. We leverage accurate information on the distribution of British plant species to infer the influence of species range overlap measured as a count of the number of 10x10km grid (hectad) overlaps. As expected, mean overlap in congeneric parental species distribution is higher for pairs of species known to give rise to hybrids (739 hectads ±27 SE, Standard Error), compared to those that have not been recorded to successfully hybridise (353 hectads ±6 SE). Our models predict significantly higher probabilities of hybrid formation when there is larger overlap in parental species distribution (pMCMC < 0.001, Appendix 2 Table 2). Although significant, the variability in the effect of parental distribution overlap is very low (posterior SD: 0.0001) compared to that of genetic distance between parental species, which is five orders of magnitude more variable (posterior SD: 3.77; Appendix 2 Figure 3). This suggests that range overlap and correlated attributes such as species abundance may be secondary to intrinsic genetic factors in determining hybridisation at a broad-spatial scale [@RN1339], though they may be key factors at a local scale [@RN1340].  In sum, there are opportunities for hybrids to occur even in areas where closely related parental species overlap little or not at all, with hybrid presence potentially affected by historical range overlap, long distance cross-pollination when parental species have coincident phenologies, or independent dispersal of hybrids [@RN1328; @RN1120].

Ploidy level variation is frequent, both within and between species in the same genus [see Chapter 2; @RN1190]. Hybrids formed from parental species of differing ploidy level (cross ploidy hybrids) appear in the British flora, with 131 detected [38% of hybridising species pairs with ploidy information @RN1007], but their relative importance has not yet been investigated across a flora. Hybrid formation is expected to decrease when the parental species have contrasting ploidy levels due to endosperm imbalance in the fertilised embryo [@RN1298]. We used ploidy data for 684 species across the British flora and determined for each pairwise comparison of species whether they were of the same or different ploidy. Our model shows that parental species with the same ploidy are 35% more likely to form hybrids than parents of differing ploidy levels, when fixed at mean overlap in geographical distribution, mean branch length between species pairs, and accounting for phylogenetic effects (pMCMC < 0.001, Appendix 2 Table 4; Appendix 2 Figures 4 and 5). Cross ploidy hybridisation has been reported in many plant genera and has led to the generation of new species [@RN1315] and introgression of genes affecting fitness [@RN980], highlighting the importance of rare hybridisation between ploidy levels.

We then investigated the impact of parental genetic distance on hybrid formation across taxa in the flora. The likelihood of hybrid formation is expected to decrease with parental genetic distance due to a greater number of genetic incompatibilities [@RN1294]. However, low genetic distances may also be a consequence of genetic homogenisation from hybridisation. We observe a ten-fold variation in mean congeneric ITS distance (see Methods for definition) across the 35 genera containing hybrids with more than five taxa, from low mean pairwise distance in *Agrostis*, *Cochlearia* and *Rosa*, to high distance in *Geranium*, *Juncus* and *Saxifraga*. Overall, hybridising congeneric species showed a significantly lower pairwise genetic distance (mean ITS distance = 0.097, SE = 0.004) than non-hybridising congeneric species pairs (mean ITS distance = 0.215, SE = 0.001, Wilcoxon Test, P<0.001; Figure \ref{CH3F2}). In our tree based phylogenetic models, the probability of forming a hybrid strongly decreases as branch length between parental species increases (pMCMC <0.001, Appendix 2 Table 2 and Appendix 2 Figure 6) and shows a greater standardised effect size than both pairwise overlap in distribution and size of genus. The stronger effect of genetic distance is evident from the joint probability distribution of hybridisation and geographical distance (Appendix 2 Figure 2), which are predicted by theory to be correlated [@RN1318]. For species which hybridise, the highest average parental genetic distance is seen in the genus *Saxifraga* (ITS2 distance: 0.28), with other divergent hybridising taxa seen in the genera *Poa* (0.22), *Cardamine* (0.19), *Potamogeton* (0.17) and *Fumaria* (0.17), showing rare examples of hybrid formation between divergent taxa. In each of these five genera, hybrids that do form tend to be sterile and therefore introgression and genetic homogenisation are unlikely [@RN1341]. This contrasts with genera characterised by low mean parental genetic distance, such as *Salicornia* (ITS2 distance: 0.00), *Prunus* (0.00), *Rosa* (0.02), *Epipactis* (0.05), and *Atriplex* (0.05), where hybrids that form tend to be fertile [@RN1007; @RN1342]. Overall these results show parental genetic divergence is a good predictor of hybrid formation, but that low parental genetic divergence may either be a cause of hybrid formation, or a consequence of genetic homogenisation. In cases where hybridisation does occur, there may be long lasting evolutionary consequences such as phenotypic novelty [@RN1316] and polyploid hybrid speciation, especially when there is high parental divergence [@RN1317].

![Hybrid formation in the context of genetic divergence in the British flora. Jittered points represent genetic distances between pairs of congeneric taxa, grouped by whether a pair of taxa produced a recorded hybrid or not. Both ITS and plastid loci are shown.\label{CH3F2}](./PDFs/Genetic_distance_boxplots2.pdf){ width=80% }

This study is the first to integrate a species level phylogeny, species traits, and ecological information for an entire flora, to understand the relative effects of genetic and ecological factors that affect hybridisation in plants. We report a high phylogenetic signal of hybridisation in flowering plants, supporting some [@RN1014] but not all previous studies [@RN1285; @RN1284], and highlighting the importance of taking species relationships into account. Phylogenetic effects are complex, and may be attributed to multiple unmeasured traits of parental species, such as habitat preferences, chromosomal stability, or mating systems [@RN1295; @RN1296; @RN1297]. Genetic distance between parental species emerges as the strongest predictor of hybridisation, emphasising the importance of this variable known to be important in pre- and postzygotic isolation [@RN1294; @RN1299]. Causality is not absolute however, as low genetic distance between parental species could also be driven by genetic homogenisation reducing sequence variability. Recently diverged species within genera such as *Euphrasia*, *Sorbus* and *Epipactis* [@RN788] are examples of genera with closely related species which hybridise extensively and where plastid and ITS introgression is likely. Hybridisation is also affected by ploidy level of parental species, with parents of differing ploidy less likely to form hybrids. Whilst ploidy differences are expected to lower the probability of hybridisation due to prezygotic barriers such as meiotic irregularities [@RN1298], the probability of forming a hybrid in this situation is still well above zero. Ploidy level can therefore be considered a leaky barrier to hybridisation [@RN311]. Overall, genetic factors at all levels have a profound impact on hybridisation.

Other, non-genetic, factors have been reported to play an important role in the prezygotic isolation of plant species and therefore in hybrid formation [@RN1302]. Factors such as genus size and life history are expected to predict hybridisation [@RN1014], however we find no evidence of this. In the case of genus size we attribute the lack of a clear cut effect to phylogenetic signal and genetic distance between species outweighing increased opportunity for hybridisation due to more congeners alone. Previous work has emphasised the importance of the perennial life history in hybridisation [@RN1120; @RN1285; @RN1284; @RN1286]. We detect no effect of life history in the probability of hybrid formation, and attribute this lack of effect to a few different factors. High recorder effort in the British Isles means that even ephemeral annual hybrids are routinely found [@RN1120], and we accounted for fewer annual than perennial plant species in the British flora by modelling all pairwise intrageneric species pairs. We also disentangled hybrid formation from hybrid persistence by using observed hybrid occurrences. Lastly, it may be that the annual genus *Euphrasia* is having a marked effect, as it is one of the few solely annual genera with many hybrids recorded [@RN1120; @RN1156]. Hybridisation is also constrained and shaped by the biogeography of parental species [@RN1303]. We reveal that a hybrid is more likely to be formed when parental species distributions are highly overlapping due to increased opportunity for crossing events; a factor not yet statistically modelled in any flora-wide study to date. Parental species overlap remains a crude estimate due to lack of resolution on fine scale co-occurrence, and does not take into account habitat change through space, or levels of habitat disturbance. Britain has a postglacial flora with high levels of disturbance which is known to change the landscape of hybridisation [@RN1304; @RN1002] and more studies are needed in undisturbed habitats to understand hybridisation in a comparative context.

## Methods

We extracted information on hybrid taxa, their parental progenitors, and ploidy from the Hybrid Flora of the British Isles [@RN1007] and used the latest plant taxonomy according to the New Flora of the British Isles [@RN1309]. The Hybrid Flora excludes complex, apomictic groups like *Hieracium*, *Taraxacum* and *Rubus*. Further ploidy information was extracted from the BSBI Cytology database and the Kew C-value database using custom python scripts (see https://github.com/Euphrasiologist/web_mining). Ploidy levels were inferred using R scripts (https://github.com/Euphrasiologist/Floristic_DNA_Barcoding) and each was checked manually, using only chromosome counts based on native UK material. Species with multiple ploidy levels were excluded unless it was exactly known which cytotype contributed to the hybrid. We excluded: (a) hybrids known not to have formed in the British Isles (e.g. taxa introduced as hybrids), (b) triple hybrids, (c) dubious or doubtful hybrids, (d) crosses at below specific rank (subspecies, varieties) and (e) hybrids where at least one parent was a recently introduced non-native species (however archaeophytes, introduced pre-1500, were included). We also removed the rare cases of intergeneric hybridisation (some Rosaceae, Poaceae and Orchidaceae) due to model scaling issues associated with looking at all possible species combinations across the flora. Downstream, hybrids were excluded if there was no barcode data associated with the parental species. Hybrid propensity was calculated by counting the total number of hybrid taxa that a particular parental taxa had participated in. We did not scale the hybrid propensity by genus size [sensu @RN1014] as the data was only used for visual interpretation. 

We estimated phylogenetic relationships from the Barcode UK dataset (Jones et al., In Review), which includes a three locus DNA barcode for British native flowering plant species. Complex or apomictic groups, as omitted from the Hybrid Flora, were not sequenced (except for *Sorbus*, as there was well sampled DNA barcodes for the group). Due to the different diversities and alignment success of plastid and nuclear ribosomal DNA we used a single alignment of plastid sequences to infer relationships between all taxa, while ITS was aligned separately for each genus and thus only used to infer congeneric relationships. Plastid DNA was aligned for all taxa using the R package DECIPHER [@Erik2016a], while ITS sequences were aligned per genus using MAFFT [@KatohandStandley2013a]. Between genera, sequences were gapped using the program catfasta2phyml (https://github.com/nylander/catfasta2phyml) using padding N's. Phylogenetic inferences were made using IQ-TREE [@RN1223] in an analysis with three partitions allowing models of molecular evolution to differ between loci, and including a multifurcating constraint tree based on APGIV relationships generated with Phylomatic [@RN1305]. Tree support was estimated using 1000 ultrafast bootstraps [@RN1306]. Tree-based genetic distances were inferred using the R function cophenetic.phylo() from the package ape [@RN1222] while separate pairwise distances for ITS and plastid DNA were calculated with the R function dist.alignment() from the seqinr package [@RN1307]. The resulting values were the square root of pairwise distances. Tree manipulation took place in R, with the circular plot made with the R package circlize [@RN1308]; the phylogeny was coerced into a circular dendrogram for visualisation. Other plots were generated with the R package ggplot2 [@Wickham2016a] and lattice [@RN1329]. All other data manipulation took place in R version 3.6.1 using base R, and packages data.table [@RN1214] and dplyr [@Wickhametal2020a].

Our final dataset for the phylogenetic analysis contained a maximum of 466 hybrid combinations in 6103 unique congeneric pairwise combinations. In the analysis of genetic distance, the plastid genetic distance dataset contained 413 hybrid combinations, and the ITS genetic distance dataset 279. We used phylogenetic generalised linear mixed models implemented in the R package MCMCglmm as it allows for addition of a phylogeny with flexible variance structures for the random effects [@RN1157]. The response variable was a binary response of whether two congeneric species produced recorded hybrids or not, and was assumed to have residuals approximated by a probit distribution. We used parameter expanded priors for better mixing and fixed the residual variance at 1. The models were run for 1.3 million iterations with a thinning interval of 1000 and a burn-in of 300000. We used five fixed effect covariates to understand their contribution to explaining the variation in hybrid formation. Firstly, pairwise branch length between parental species calculated from the phylogeny (above) was added to understand the contribution of intrageneric relatedness. Pairwise overlap in geographical distribution (number of 10x10km2; generated from data at https://database.bsbi.org/) accounted for extent of overlap of parental species. Genus size for each species was calculated from species present in the phylogeny. Lastly, whether the parental species were of the same ploidy level or not, was added as a two level categorical factor; same ploidy level (homoploid) or not (heteroploid). We ran two models which differed in their fixed effect structure only, with the addition of the ploidy level and one without. This was because ploidy data was limited either due to missing counts or if ploidy was difficult to estimate; the number of species analysed decreased from 1098 to 684 upon addition of ploidy level. 148 out of 244 families contained missing ploidy data, with a few genera containing high amounts of missing data, e.g. *Sorbus*, *Alchemilla*, *Juncus*, and especially *Carex*. The phylogenetic (species level) best linear unbiased predictors (BLUP’s; means of the posterior distribution) were extracted from the model excluding ploidy level and are equivalent to the per species point estimates of the probability of hybridisation.

The inverse relatedness matrix (unscaled phylogeny) and species were fitted as random effects in a multi-membership model structure, as each hybrid event is the outcome of two parental species. The effect of phylogeny was added to the fixed effect predictions by calculating:

\begin{equation}
\sigma_{u}^2 + 2\sigma_{s}^2 + (\sigma_{p}^2)\overline{B}
\end{equation}

Where *u* is the residual variance, *s* is the species variance and *p* is the phylogenetic variance. $\overline{B}$ is defined as the average tree of all species pairs:

\begin{equation}
\overline{B} =  2\overline{d} + 2\overline{o}
\end{equation}

Where $\overline{d}$ is the average species phylogenetic variance and $\overline{o}$ is the sum of the pairwise species phylogenetic covariances divided by the number of possible combinations of species multiplied by two. As we did not allow intergeneric hybridisation, we calculated $\overline{B}$ at the species level constrained to genus ($\overline{B}_{S}$):

\begin{equation}
\overline{B}_{S} = \frac{2\sum o_{i}nc_{i} + 2\sum d_{i}n_{i}}{\sum nc_{i}}
\end{equation}

*i* represents the *i*th genus. Variables *o* and *d* are defined as above but calculated for each genus, $n_{i}$ is the number of individuals in a genus and $nc_{i}$ is the number of possible pairwise combinations within genus. Using this method, we were able to account for the size of genus in our phylogenetic variance estimates. We implement this algorithm in the R package VCVglmm [@Brown2020a]. Lastly, phylogenetic signal was calculated using:

\begin{equation}
\frac{\sigma_{p}^2\overline{B}_{S}}{\sigma_{u}^2 + 2\sigma_{s}^2 + 2\sigma_{p}^2\overline{B}_{S}}
\end{equation}

All parameters are sampled from the posterior distribution of each coefficient, and distributions are summarised using modes and highest posterior density intervals at the 95% level. P-values were taken directly from model output for continuous covariates or categorical covariates with only two levels, otherwise (for life history effects) Wald Tests jointly tested all factor levels [@Brown2020a]. Genetic distance comparisons were made using Wilcoxon tests on the pairwise genetic distances, which does not assume that the distances approximate any distribution.
